icons/nano/nano-original-wordmark.svg

```diff
@@ -1 +1,2774 @@
354 0 0 0 .257.097.351.351 0 0 0 .258-.098.334.334 0 0 0 .1-.251.323.323 0 0 0-.1-.244c-.067-.064-.153-.096-.258-.096s-.191.032-.258.096a.334.334 0 0 0-.099.247zm1.935.449a.559.559 0 0 1-.224-.186.464.464 0 0 1-.075-.263c0-.159.057-.289.17-.391a.626.626 0 0 1 .435-.153c.174 0 .318.051.434.153a.48.48 0 0 1 .173.375.503.503 0 0 1-.074.278.534.534 0 0 1-.228.187.519.519 0 0 1 .238.185.496.496 0 0 1 .083.286.533.533 0 0 1-.174.412.645.645 0 0 1-.452.159.639.639 0 0 1-.451-.161.54.54 0 0 1-.173-.417.484.484 0 0 1 .318-.464zm-.065.464c0 .108.034.196.102.264a.37.37 0 0 0 .271.101.378.378 0 0 0 .272-.098.344.344 0 0 0 .102-.261c0-.106-.035-.192-.104-.258s-.159-.099-.27-.099c-.109 0-.199.032-.268.097s-.105.15-.105.254zm.015-.913a.33.33 0 0 0 .1.249.354.354 0 0 0 .257.097.351.351 0 0 0 .258-.098.334.334 0 0 0 .1-.251.323.323 0 0 0-.1-.244.358.358 0 0 0-.258-.096.358.358 0 0 0-.258.096.33.33 0 0 0-.099.247zm3.838.096c-.026 0-.044-.006-.056-.019-.012-.013-.019-.039-.022-.077l-.048-.566v-.018c0-.026.006-.044.017-.057.011-.012.029-.018.052-.018h.147c.024 0 .041.006.052.018s.017.031.017.057v.018l-.046.566c-.004.039-.012.065-.023.077s-.03.019-.055.019h-.035zm.542 0c-.026 0-.044-.006-.055-.019s-.019-.039-.023-.077l-.048-.566v-.018c0-.026.006-.044.018-.057.012-.012.029-.018.051-.018h.149c.022 0 .04.006.051.018.012.012.018.031.018.057v.018l-.048.566c-.004.039-.012.065-.023.077s-.03.019-.055.019h-.035zm1.324.353a.559.559 0 0 1-.224-.186.464.464 0 0 1-.075-.263c0-.159.057-.289.17-.391a.626.626 0 0 1 .435-.153c.174 0 .318.051.434.153a.48.48 0 0 1 .173.375.503.503 0 0 1-.074.278.534.534 0 0 1-.228.187.519.519 0 0 1 .238.185.496.496 0 0 1 .083.286.533.533 0 0 1-.174.412.645.645 0 0 1-.452.159.639.639 0 0 1-.451-.161.54.54 0 0 1-.173-.417.484.484 0 0 1 .318-.464zm-.066.464c0 .108.034.196.102.264a.37.37 0 0 0 .271.101.378.378 0 0 0 .272-.098.344.344 0 0 0 .102-.261.341.341 0 0 0-.104-.258.376.376 0 0 0-.27-.099c-.109 0-.199.032-.268.097s-.105.15-.105.254zm.015-.913a.33.33 0 0 0 .1.249.354.354 0 0 0 .257.097.351.351 0 0 0 .258-.098.334.334 0 0 0 .1-.251.323.323 0 0 0-.1-.244.358.358 0 0 0-.258-.096.358.358 0 0 0-.258.096.33.33 0 0 0-.099.247zm1.935.449a.559.559 0 0 1-.224-.186.464.464 0 0 1-.075-.263c0-.159.057-.289.17-.391a.626.626 0 0 1 .435-.153c.174 0 .318.051.434.153a.48.48 0 0 1 .173.375.503.503 0 0 1-.074.278.534.534 0 0 1-.228.187.519.519 0 0 1 .238.185.496.496 0 0 1 .083.286.533.533 0 0 1-.174.412c-.116.106-.267.159-.452.159s-.335-.054-.451-.161a.54.54 0 0 1-.173-.417.484.484 0 0 1 .318-.464zm-.066.464c0 .108.034.196.102.264a.37.37 0 0 0 .271.101.378.378 0 0 0 .272-.098.344.344 0 0 0 .102-.261.341.341 0 0 0-.104-.258.376.376 0 0 0-.27-.099.376.376 0 0 0-.268.097.331.331 0 0 0-.105.254zm.015-.913a.33.33 0 0 0 .1.249.354.354 0 0 0 .257.097.351.351 0 0 0 .258-.098.334.334 0 0 0 .1-.251.323.323 0 0 0-.1-.244c-.067-.064-.153-.096-.258-.096s-.191.032-.258.096a.334.334 0 0 0-.099.247zm1.846.244a.555.555 0 0 1 .215-.164.713.713 0 0 1 .291-.057.703.703 0 0 1 .719.722c0 .20
```

Comment:
Should be minified using [SVGOMG](https://jakearchibald.github.io/svgomg/)
I went ahead and did it for you, so all you have to do is click apply:)

```suggestion
<svg xmlns="http://www.w3.org/2000/svg" xml:space="preserve" viewBox="0 0 128 128">
  <path fill="#c8f" d="M32.5 31.5c0-.1 0-.1.1-.2s.1-.1.2-.1.2 0 .2.1c.1 0 .1.1.1.2s0 .1-.1.2-.1.1-.2.1-.2 0-.2-.1-.1-.1-.1-.2zm0 .9c0-.1 0-.1.1-.2s.1-.1.2-.1.2 0 .2.1.1.1.1.2 0 .1-.1.2-.1.1-.2.1-.2 0-.2-.1-.1-.1-.1-.2zm1.9-.9c0-.1 0-.1.1-.2s.1-.1.2-.1.2 0 .2.1.1.1.1.2 0 .1-.1.2-.1.1-.2.1-.2 0-.2-.1-.1-.1-.1-.2zm0 .9c0-.1 0-.1.1-.2s.1-.1.2-.1.2 0 .2.1.1.1.1.2 0 .1-.1.2-.1.1-.2.1-.2 0-.2-.1-.1-.1-.1-.2zm1.9-.9c0-.1 0-.1.1-.2s.1-.1.2-.1.2 0 .2.1c.1 0 .1.1.1.2s0 .1-.1.2-.1.1-.2.1-.2 0-.2-.1c-.1 0-.1-.1-.1-.2zm0 .9c0-.1 0-.1.1-.2s.1-.1.2-.1.2 0 .2.1.1.1.1.2 0 .1-.1.2-.1.1-.2.1-.2 0-.2-.1c-.1-.1-.1-.1-.1-.2zM6.6 36.2h.6v.2H5.8v-.2h.6v-1H6V35h.6v1.2zm-.3-1.7v-.2h.2v.4c0 .1-.1 0-.1 0h-.1v-.2zm2.6 1.3v-.1h.2V36.5H7.5v-.2H7.8v-1.4h-.2v-.2h1v.2h-.5v1.4h.8v-.5zm.7.6h-.2v-.2H9.7v-1.4h-.2v-.2H11v.6h-.2v-.4h-.9v.5h.4v-.2h.2v.5h-.2v-.2h-.4v.7h.8v-.4h.2v.6h-.1l-1.2.1zm2.2-1c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4s.3-.2.4-.2c.2 0 .3.1.4.2s.2.2.2.4c0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4s-.3.2-.5.2-.3-.1-.5-.2-.2-.2-.2-.4c.2-.2.3-.4.5-.5zm-.1.5c0 .1 0 .2.1.3s.2.1.3.1.2 0 .3-.1.1-.2.1-.3c0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1 0 .1-.1.2-.1.3zm.1-.9c0 .1 0 .2.1.2s.2.1.3.1c.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1-.1.1-.2.2-.1.3zm1.9.4c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.1.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2s.2.2.2.4c0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4s-.3.2-.5.2-.3-.1-.5-.2-.2-.2-.2-.4c.2-.2.3-.4.5-.5zm-.1.5c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1 0 .1-.1.2-.1.3zm0-.9c0 .1 0 .2.1.2s.2.1.3.1c.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1s-.2 0-.3.1c0 .1-.1.2-.1.3zm1.9 1.4h-.4v-.2h.1v-1.4h-.1v-.2h.7c.3 0 .5.1.6.2.2.2.2.4.2.7 0 .1 0 .3-.1.4 0 .1-.1.2-.2.3-.1.1-.2.1-.3.2-.1 0-.3.1-.4.1h-.1c.1-.1 0-.1 0-.1zm0-.2h.2c.3 0 .4-.1.6-.2.1-.1.2-.3.2-.5s-.1-.4-.2-.5c-.1-.1-.3-.2-.6-.2h-.2v1.4zm2.7.1v.3c0 .1 0 .1-.1.2l-.2.2c-.1 0-.2.1-.4.1h-.3c-.1 0-.1-.1-.1-.1v-.1H17.4c.1 0 .3 0 .3-.1.1-.1.1-.2.1-.4v-1.1H17v-.2h1l.2 1.2zm-.4-1.8v-.2h.2v.4h-.2v-.2zm1.5 1.6c0-.1 0-.1.1-.2s.1-.1.2-.1.2 0 .2.1.1.1.1.2 0 .1-.1.2-.1.1-.2.1-.2 0-.2-.1c0 0-.1-.1-.1-.2zm5.7-.8c0-.1 0-.1.1-.2s.1-.1.2-.1.2 0 .2.1.1.1.1.2 0 .1-.1.2-.1.1-.2.1-.2 0-.2-.1c-.1-.1-.1-.1-.1-.2zm0 .8c0-.1 0-.1.1-.2s.1-.1.2-.1.2 0 .2.1.1.1.1.2 0 .1-.1.2-.1.1-.2.1-.2 0-.2-.1c-.1 0-.1-.1-.1-.2zm2.7.2v.3c0 .1 0 .1-.1.2l-.2.2c-.1 0-.2.1-.4.1h-.3c-.1 0-.1-.1-.1-.1v-.1H26.9c.1 0 .3 0 .3-.1.1-.1.1-.2.1-.4v-1.1h-.8v-.2h1c.2-.1.2 1.2.2 1.2zm-.5-1.8v-.2h.2v.4h-.2v-.2zm1.5 1.9h-.4v-.2h.1v-1.4h-.1v-.2h.7c.3 0 .5.1.6.2.2.2.2.4.2.7 0 .1 0 .3-.1.4 0 .1-.1.2-.2.3-.1.1-.2.1-.3.2-.1 0-.3.1-.4.1h-.1c.1-.1 0-.1 0-.1zm0-.2h.2c.3 0 .4-.1.6-.2.1-.1.2-.3.2-.5s-.1-.4-.2-.5c-.1-.1-.3-.2-.6-.2h-.2v1.4zm1.9-.8c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4s.3-.2.4-.2c.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4l.5-.5zm0 .5c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1-.1.1-.1.2-.1.3zm0-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1-.1.1-.1.2-.1.3zm1.9.4c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4s.3-.2.4-.2c.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.2-.2.3-.4.5-.5zm0 .5c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1s.1-.2.1-.3c0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1-.1.1-.1.2-.1.3zm0-.9c0 .1 0 .2.1.2s.2.1.3.1c.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1-.1.1-.1.2-.1.3zm1.9.4c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.1.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2s.2.2.2.4c0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.2-.2.3-.4.5-.5zm-.1.5c0 .1 0 .2.1.3s.2.1.3.1.2 0 .3-.1.1-.2.1-.3c0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1 0 .1-.1.2-.1.3zm.1-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1s-.2 0-.3.1c-.1.1-.1.2-.1.3zm1.9.4c-.1 0-.2-.1-.2-.2-.1-.1-.1-.1-.1-.2s.1-.3.2-.4c.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4s-.3.2-.5.2-.3-.1-.5-.2 0-.4 0-.5c0-.2.1-.4.3-.5zm-.1.5c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1 0 .1-.1.2-.1.3zm0-.9c0 .1 0 .2.1.2s.2.1.3.1c.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1 0 .1-.1.2-.1.3zm2 .4c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4s.3-.2.4-.2c.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.2-.2.3-.4.5-.5zm-.1.5c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1s.1-.2.1-.3c0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1 0 .1-.1.2-.1.3zm0-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1 0 .1-.1.2-.1.3zm1.9 1.4h-.4v-.2h.1v-1.4h-.1v-.2h.7c.3 0 .5.1.6.2.2.2.2.4.2.7 0 .1 0 .3-.1.4 0 .1-.1.2-.2.3-.1.1-.2.1-.3.2-.1 0-.3.1-.4.1H40c.1-.1 0-.1 0-.1zm0-.2h.2c.3 0 .4-.1.6-.2.1-.1.2-.3.2-.5s-.1-.4-.2-.5c-.1-.1-.3-.2-.6-.2H40v1.4zm2.7.1v.3c0 .1 0 .1-.1.2l-.2.2h-.7c-.1 0-.1-.1-.1-.1v-.1H41.9c.1 0 .3 0 .3-.1.1-.1.1-.2.1-.4v-1.1h-.8V35h1v1.3zm-.4-1.8v-.2h.2v.4h-.2v-.2zm1.5.8c0-.1 0-.1.1-.2s.1-.1.2-.1.2 0 .2.1.1.1.1.2 0 .1-.1.2-.1.1-.2.1-.2 0-.2-.1-.1-.1-.1-.2zm0 .8c0-.1 0-.1.1-.2s.1-.1.2-.1.2 0 .2.1.1.1.1.2 0 .1-.1.2-.1.1-.2.1-.2 0-.2-.1-.1-.1-.1-.2zM2.4 39.9c0-.1 0-.1.1-.2s.1-.1.2-.1.2 0 .2.1.1.1.1.2 0 .1-.1.2c-.1 0-.1.1-.2.1s-.2 0-.2-.1-.1-.1-.1-.2zm2.7-.4v-.1h.2V40.2H3.7V40H4v-1.4h-.2v-.2h1v.2h-.5V40h.8v-.5zm1.8.4v-.4h-.4v-.2H7.4c.1 0 0 0 0 .1v.1h-.1v.6c-.1 0-.2.1-.3.1h-.3c-.3 0-.5-.1-.7-.3s-.3-.4-.3-.7c0-.3.1-.5.2-.7.2-.2.4-.3.6-.3h.2c.1 0 .1 0 .2.1v-.1h.2v.5H7c-.1 0-.1 0-.1-.1-.1-.2-.2-.3-.4-.3s-.3.1-.4.2c-.1.3-.2.5-.2.7 0 .2.1.4.2.6.1.1.3.2.5.2h.2l.1-.1zm1.6.1h.6v.2H7.7V40h.6v-1h-.5v-.2h.6c.1 0 .1 1.2.1 1.2zm-.3-1.7v-.2h.2v.4c0 .1-.1 0-.1 0h-.1v-.2zm2.4.5h.1v.2H10v.5c0 .2 0 .3.1.3s.1.1.2.1.2 0 .3-.1c.1 0 .2-.1.2-.1h.1v.2s-.1 0-.1.1c-.1 0-.2.1-.3.1h-.2c-.1 0-.2 0-.3-.1 0 .1-.1 0-.1 0v-.9h-.5v-.2h.4v-.5h.2v.5l.6-.1zm.9 1.4h-.2c-.1 0 0 0 0-.1V40H11.6v-1.4h-.2v-.2H12.9v.6h-.2v-.4h-.8v.5h.4v-.2h.2v.5h-.2v-.2h-.4v.7h.8v-.4h.2v.6h-.1l-1.3.1zm2.2-1c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.1.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4s-.3.2-.5.2-.3-.1-.5-.2-.2-.2-.2-.4c.2-.3.3-.5.5-.5zm-.1.4c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1 0 .2-.1.2-.1.3zm0-.9c0 .1 0 .2.1.2s.2.1.3.1c.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2s-.2-.1-.3-.1-.2 0-.3.1-.1.2-.1.3zm2 .5c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.1.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2-.2-.2-.2-.4c.1-.3.3-.5.5-.5zm-.1.4c0 .1 0 .2.1.3s.2.1.3.1c.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1-.1.2-.1.2-.1.3zm0-.9c0 .1 0 .2.1.2s.2.1.3.1c.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1s-.1.2-.1.3zm2 .5c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.1.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.1-.3.3-.5.5-.5zm-.1.4c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3s-.2-.1-.3-.1c-.1 0-.2 0-.3.1s-.1.2-.1.3zm0-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1s-.2 0-.3.1c-.1.1-.1.2-.1.3zm1.9 1.5h-.4V40h.1v-1.4h-.1v-.2h.7c.3 0 .5.1.6.2.2.2.2.4.2.7 0 .1 0 .3-.1.4 0 .1-.1.2-.2.3-.1.1-.2.1-.3.2s-.3.1-.4.1h-.1c.1-.1 0-.1 0-.1zm0-.2h.2c.3 0 .4-.1.6-.2.1-.1.2-.3.2-.5s-.1-.4-.2-.5c-.1-.1-.3-.2-.6-.2h-.2V40zm1.9-.1c0-.1 0-.1.1-.2s.1-.1.2-.1.2 0 .2.1.1.1.1.2 0 .1-.1.2-.1.1-.2.1-.2 0-.2-.1c0 0-.1-.1-.1-.2zm2.7 0s.1.1 0 0c.1.1.1.1.1.2v.1H22.8V40h.4v-1h-.4v-.2H23.1v-.1c0-.2.1-.3.2-.4.1-.1.3-.1.5-.1h.3c.1 0 .1.1.1.1v.1h-.4c-.1 0-.2 0-.2.1s-.1.1-.1.2v.1h.6v.2h-.6v.9h.4zm1.1-.7c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.1.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.2-.3.3-.5.5-.5zm-.1.4c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3s-.2-.1-.3-.1c-.1 0-.2 0-.3.1 0 .2-.1.2-.1.3zm0-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1 0 .1-.1.2-.1.3zm2.8 1.2v-.4h-.5v-.2H28.1c.1 0 0 0 0 .1v.1H28v.6c-.1 0-.1.1-.3.1h-.3c-.3 0-.5-.1-.7-.3-.2-.2-.3-.4-.3-.7 0-.3.1-.5.2-.7.2-.2.4-.3.6-.3h.2c.1 0 .1 0 .2.1v-.1h.2v.5h-.1c-.1 0-.1 0-.1-.1-.1-.2-.2-.3-.4-.3s-.3.1-.4.2c-.1.1-.2.3-.2.5s.1.4.2.6c.1.1.3.2.5.2h.2c0 .2.1.1.2.1zm1.8.2v.3c0 .1 0 .1-.1.2l-.2.2c-.1 0-.2.1-.4.1h-.3c-.1 0-.1-.1-.1-.1v-.1H28.7c.1 0 .3 0 .3-.1s.1-.2.1-.4V39h-.8v-.2h1c.2 0 .2 1.3.2 1.3zm-.4-1.8v-.2h.2v.4h-.2v-.2zm2.3 1.8v.3c0 .1 0 .1-.1.2l-.2.2c-.1 0-.2.1-.4.1h-.3c-.1 0-.1-.1-.1-.1v-.1H30.6c.1 0 .3 0 .3-.1s.1-.2.1-.4V39h-.8v-.2h1c.2 0 .2 1.3.2 1.3zm-.4-1.8v-.2h.2v.4H31v-.2zm2.3 1.8v.3c0 .1 0 .1-.1.2l-.2.2c-.1 0-.2.1-.4.1h-.3c-.1 0-.1-.1-.1-.1v-.1H32.5c.1 0 .3 0 .3-.1s.1-.2.1-.4V39h-.8v-.2h1v1.3zm-.4-1.8v-.2h.2v.4h-.2v-.2zm2.4 1.3v-.1h.2V40.3h-1.6v-.2H34.2v-1.4H33.9v-.2h1v.2h-.5V40h.8l.1-.4zm1-.4c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.1.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4s-.3.2-.5.2-.3-.1-.5-.2 0-.3 0-.5.1-.4.3-.4zm-.1.4c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1 0 .2-.1.2-.1.3zm0-.9c0 .1 0 .2.1.2s.2.1.3.1c.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2s-.2-.1-.3-.1-.2 0-.3.1-.1.2-.1.3zm2 .5c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.1.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.2-.3.3-.5.5-.5zm-.1.4c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3s-.2-.1-.3-.1c-.1 0-.2 0-.3.1s-.1.2-.1.3zm0-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1 0 .1-.1.2-.1.3zm2 .5c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.1.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.1-.3.3-.5.5-.5zm-.1.4c0 .1 0 .2.1.3s.2.1.3.1c.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3s-.2-.1-.3-.1c-.1 0-.2 0-.3.1s-.1.2-.1.3zm0-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2s-.2-.1-.3-.1-.2 0-.3.1c-.1.1-.1.2-.1.3zm2 .5c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.1.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4s-.3.2-.5.2-.3-.1-.5-.2-.2-.2-.2-.4c.1-.3.3-.5.5-.5zm-.1.4c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1-.1.2-.1.2-.1.3zm0-.9c0 .1 0 .2.1.2s.2.1.3.1c.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2s-.2-.1-.3-.1-.2 0-.3.1-.1.2-.1.3zm1.6 1.5h-.2V40H43.6v-1.4h-.2v-.2H44.9v.6h-.2v-.4h-.8v.5h.4v-.2h.2v.5h-.2v-.2h-.4v.7h.8v-.4h.2v.6h-.1l-1.3.1zm2.3-.4s.1-.1 0 0c.1-.1.1-.1 0 0l.4-.1-.4.8h-.1c-.1 0 .1-.7.1-.7zm-.1-.7c0-.1 0-.1.1-.2s.1-.1.2-.1.2 0 .2.1.1.1.1.2 0 .1-.1.2-.1.1-.2.1-.2 0-.2-.1-.1-.1-.1-.2zm17 .8c0-.1 0-.1.1-.2s.1-.1.2-.1.2 0 .2.1.1.1.1.2 0 .1-.1.2-.1.1-.2.1-.2 0-.2-.1c-.1 0-.1-.1-.1-.2zm2.6.3c0-.1 0-.1 0 0V40l-.2.2c-.1.1-.2.1-.3.1-.2 0-.4-.1-.5-.2-.1-.1-.2-.3-.2-.5 0-.4.3-.7.7-.7.1 0 .2 0 .3.1.1 0 .2.1.2.2v-.6H65v-.2h.5V40h.2v.2h-.4zm-1-.7c0 .1 0 .3.1.3s.2.1.3.1.3 0 .3-.1.1-.2.1-.4c0-.3-.2-.5-.4-.5-.1 0-.3 0-.3.1s-.1.3-.1.5zm2.1-.3c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.1.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.2-.3.4-.5.5-.5zm0 .4c0 .1 0 .2.1.3s.2.1.3.1c.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1-.1.2-.1.2-.1.3zm0-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1s-.2 0-.3.1c-.1.1-.1.2-.1.3zm1.9.5c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.1.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.2-.3.3-.5.5-.5zm0 .4c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1-.1.2-.1.2-.1.3zm0-.9c0 .1 0 .2.1.2s.2.1.3.1c.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1s-.2 0-.3.1c-.1.1-.1.2-.1.3zm1.9.5c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.1.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.2-.3.3-.5.5-.5zm0 .4c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1-.1.2-.2.2-.1.3zm0-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1s-.1.2-.1.3zm1.9.5c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.1.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.2-.3.3-.5.5-.5zm-.1.4c0 .1 0 .2.1.3s.2.1.3.1c.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1 0 .2-.1.2-.1.3zm0-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1s-.2 0-.3.1c0 .1-.1.2-.1.3zm1.9.3.2-.2c.1 0 .2-.1.3-.1.4 0 .7.3.7.7 0 .2-.1.4-.2.5-.1.1-.3.2-.5.2-.1 0-.2 0-.3-.1-.1.1-.1 0-.2 0v.2c0 .1 0 0 0 0h-.4V40h.1v-1.6h-.1v-.2h.4v.8zm.5 1c.1 0 .3 0 .4-.1.1-.1.1-.2.1-.4 0-.3-.2-.5-.5-.5-.1 0-.3 0-.3.1-.1.1-.1.2-.1.4s0 .3.1.4c0 0 .1.1.3.1zm1.5-.1c0-.1 0-.1.1-.2s.1-.1.2-.1.2 0 .2.1.1.1.1.2 0 .1-.1.2-.1.1-.2.1-.2 0-.2-.1-.1-.1-.1-.2zm5.6-.7c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.1.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.2-.3.3-.5.5-.5zm0 .4c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3s-.2-.1-.3-.1c-.1 0-.2 0-.3.1s-.1.2-.1.3zm0-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1-.1.1-.1.2-.1.3zm1.9.5c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.1.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4s-.3.2-.5.2-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.2-.3.3-.5.5-.5zm-.1.4c0 .1 0 .2.1.3s.2.1.3.1c.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1 0 .2-.1.2-.1.3zm.1-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2s-.2-.1-.3-.1-.2 0-.3.1c-.1.1-.1.2-.1.3zm1.9.5c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.1.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2v-.5c0-.2.1-.4.3-.4zm-.1.4c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1 0 .2-.1.2-.1.3zm0-.9c0 .1 0 .2.1.2s.2.1.3.1c.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1s-.2 0-.3.1c0 .1-.1.2-.1.3zm1.9.3.2-.2c.1 0 .2-.1.3-.1.4 0 .7.3.7.7 0 .2-.1.4-.2.5-.1.1-.3.2-.5.2-.1 0-.2 0-.3-.1-.1 0-.1-.1-.2-.2v.2c0 .1 0 0 0 0h-.4v-.2h.1v-1.6h-.1V38h.4v1zm.5 1c.1 0 .3 0 .4-.1.1-.1.1-.2.1-.4 0-.3-.2-.5-.5-.5-.1 0-.3 0-.3.1-.1.1-.1.2-.1.4s0 .3.1.4c0 0 .1.1.3.1zm9-.8c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.1.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.2-.3.3-.5.5-.5zm-.1.4c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1 0 .2-.1.2-.1.3zm0-.9c0 .1 0 .2.1.2s.2.1.3.1c.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1s-.2 0-.3.1c0 .1-.1.2-.1.3zm2 .5c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.1.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.2-.3.3-.5.5-.5zm-.1.4c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3s-.2-.1-.3-.1c-.1 0-.2 0-.3.1s-.1.2-.1.3zm0-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1 0 .1-.1.2-.1.3zm2 .5c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.1.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4s-.3.2-.5.2-.3-.1-.5-.2-.2-.2-.2-.4c.1-.3.3-.5.5-.5zm-.1.4c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1-.1.2-.1.2-.1.3zm0-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2s-.2-.1-.3-.1-.2 0-.3.1-.1.2-.1.3zm3.8.5c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.1.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.2-.3.3-.5.5-.5zm0 .4c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1s-.1.2-.1.3zm0-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1-.1.1-.1.2-.1.3zm1.9.5c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.1.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4s-.3.2-.5.2-.3-.1-.5-.2-.2-.2-.2-.4c.2-.3.3-.5.5-.5zm0 .4c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1-.1.2-.1.2-.1.3zm0-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2s-.2-.1-.3-.1-.2 0-.3.1-.1.2-.1.3zm1.9.5c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.1.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.2-.3.3-.5.5-.5zm-.1.4c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1 0 .2-.1.2-.1.3zm.1-.9c0 .1 0 .2.1.2s.2.1.3.1c.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1s-.2 0-.3.1c-.1.1-.1.2-.1.3zm11.3.5c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.1.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.2-.3.3-.5.5-.5zm-.1.4c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1 0 .2-.1.2-.1.3zm.1-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1s-.2 0-.3.1c-.1.1-.1.2-.1.3zm1.9.5c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.1.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.2-.3.3-.5.5-.5zm-.1.4c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1 0 .2-.1.2-.1.3zm0-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1 0 .1-.1.2-.1.3zm2 .5c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.1.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4s-.3.2-.5.2-.3-.1-.5-.2-.2-.2-.2-.4c.2-.3.3-.5.5-.5zm-.1.4c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1-.1.2-.1.2-.1.3zm0-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2s-.2-.1-.3-.1-.2 0-.3.1-.1.2-.1.3zm-120.1 5h.6v.2H2v-.2h.6v-1h-.4v-.2h.6v1.2zm-.3-1.6v-.2h.2v.4c0 .1-.1 0-.1 0h-.1c.1-.1 0-.2 0-.2zm1.4 1.8h-.2v-.2H4v-1.4h-.2v-.2H5.3v.6h-.2v-.4h-.8v.5h.4v-.2h.2v.5h-.2V43h-.4v.7h.8v-.4c0-.1 0 0 .1 0h.1v.6H3.9zm7.9-1.1c0-.1 0-.1.1-.2s.1-.1.2-.1.2 0 .2.1.1.1.1.2 0 .1-.1.2-.1.1-.2.1-.2 0-.2-.1-.1-.1-.1-.2zm0 .9c0-.1 0-.1.1-.2s.1-.1.2-.1.2 0 .2.1.1.1.1.2 0 .1-.1.2-.1.1-.2.1-.2 0-.2-.1c-.1-.1-.1-.1-.1-.2zm1.9-.8c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.1.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2s.2.2.2.4c0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4s-.2.2-.4.2-.3-.1-.5-.2-.2-.2-.2-.4c.1-.2.2-.4.4-.5zm-.1.5c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1 0 .1-.1.2-.1.3zm0-.9c0 .1 0 .2.1.2s.2.1.3.1c.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1s-.2 0-.3.1c0 .1-.1.2-.1.3zm2 .4c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.1.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2-.2-.2-.2-.4c.1-.2.3-.4.5-.5zm-.1.5c0 .1 0 .2.1.3s.2.1.3.1c.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1-.1.1-.1.2-.1.3zm0-.9c0 .1 0 .2.1.2s.2.1.3.1c.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1 0 .1-.1.2-.1.3zm2 .4c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4s.3-.2.4-.2c.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.1-.2.3-.4.5-.5zm-.1.5c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1-.1.1-.1.2-.1.3zm0-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1s-.2 0-.3.1c-.1.1-.1.2-.1.3zm1.9.4c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4s.3-.2.4-.2c.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4 0 .1-.2.2-.4.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.1-.2.2-.4.4-.5zm0 .5c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1s.1-.2.1-.3c0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1-.1.1-.1.2-.1.3zm0-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1-.1.1-.1.2-.1.3zm1.6 1.4h-.2v-.2H21v-1.4h-.2v-.2H22.3v.6h-.2v-.4h-.8v.5h.4v-.2h.2v.5h-.2V43h-.4v.7h.7v-.4h.2v.6h-1.3zm2.9-1.3h.1v.2h-.7v.5c0 .2 0 .3.1.3 0 .1.1.1.2.1s.2 0 .3-.1c.1 0 .2-.1.2-.1h.1v.2s-.1 0-.1.1c-.1 0-.2.1-.3.1h-.2c-.1 0-.2 0-.3-.1-.1 0-.1-.1-.2-.2v-.9h-.4v-.2h.4V42h.2v.5l.6.1zm1.2 1.1c0-.1 0-.1.1-.2s.1-.1.2-.1.2 0 .2.1.1.1.1.2 0 .1-.1.2-.1.1-.2.1-.2 0-.2-.1c-.1-.1-.1-.1-.1-.2zm11.3 0c0-.1 0-.1.1-.2s.1-.1.2-.1.2 0 .2.1.1.1.1.2 0 .1-.1.2-.1.1-.2.1-.2 0-.2-.1c-.1-.1-.1-.1-.1-.2zm2.7 0v-.4h-.5v-.2H39.4v.2h-.1v.6c-.1 0-.1.1-.3.1h-.3c-.3 0-.5-.1-.7-.3s-.3-.4-.3-.7c0-.3.1-.5.2-.7.2-.2.4-.3.6-.3h.2c.1 0 .1 0 .2.1V42h.2v.5H39c-.1 0-.1 0-.1-.1-.1-.2-.2-.3-.4-.3s-.3.1-.4.2c-.1.1-.2.3-.2.5s.1.4.2.6.3.2.5.2h.2c0 .1.1.1.2.1zm1.1-.8c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.1.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2s.2.2.2.4c0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.1-.2.3-.4.5-.5zm-.1.5c0 .1 0 .2.1.3s.2.1.3.1c.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1-.1.1-.1.2-.1.3zm0-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1s-.2 0-.3.1c-.1.1-.1.2-.1.3zm2 .4c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.1.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4s-.3.2-.4.2-.3-.1-.5-.2-.2-.2-.2-.4.2-.4.4-.5zm-.1.5c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1-.1.1-.1.2-.1.3zm0-.9c0 .1 0 .2.1.2s.2.1.3.1c.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1-.1.1-.1.2-.1.3zm1.9.4c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4s.3 0 .4 0c.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.2-.4.3-.6.5-.7zm0 .5c0 .1 0 .2.1.3s.2.1.3.1.2 0 .3-.1.1-.2.1-.3c0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1-.1.1-.1.2-.1.3zm0-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1s-.2 0-.3.1c-.1.1-.1.2-.1.3zm1.9.4c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.1.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2s.2.2.2.4c0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4s-.2.2-.4.2-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.1-.2.2-.4.4-.5zm0 .5c0 .1 0 .2.1.3s.2.1.3.1c.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1-.1.1-.2.2-.1.3zm0-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1-.1.1-.1.2-.1.3zm1.9 1.2c0-.1 0-.1.1-.2s.1-.1.2-.1.2 0 .2.1.1.1.1.2 0 .1-.1.2c0 0-.1.1-.2.1s-.2 0-.2-.1c-.1-.1-.1-.1-.1-.2zm14 .2c-.1 0-.1 0 0 0-.1 0-.1 0 0 0l-.1-.2-.2.2c-.1.1-.2.1-.3.1-.2 0-.4-.1-.5-.2-.1-.1-.2-.3-.2-.5 0-.4.3-.7.7-.7.1 0 .2 0 .3.1.1 0 .2.1.2.2v-.6H61.2c-.1 0 0 0 0-.1v-.1h.5v1.7h.2v.2h-.1l-.2-.1zm-1-.7c0 .1 0 .3.1.3.1.1.2.1.3.1.1 0 .3 0 .3-.1.1-.1.1-.2.1-.4 0-.3-.2-.5-.4-.5-.1 0-.3 0-.3.1-.1.3-.1.4-.1.5zm2.1-.3c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.1.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2s.2.2.2.4c0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4s-.2.2-.4.2-.3-.1-.5-.2-.2-.2-.2-.4c.1-.2.2-.4.4-.5zm-.1.5c0 .1 0 .2.1.3s.2.1.3.1c.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1-.1.1-.1.2-.1.3zm0-.9c0 .1 0 .2.1.2s.2.1.3.1c.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1 0 .1-.1.2-.1.3zm2 .4c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.1.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.4.2-.2 0-.3-.1-.5-.2s-.2-.2-.2-.4.2-.4.4-.5zm-.1.5c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1-.1.1-.1.2-.1.3zm0-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1-.1.1-.1.2-.1.3zm2.3 1.2h.1v.2h-1v-.2H66.2v-1.4H65.9v-.2H66.7c.2 0 .4 0 .6.1s.2.2.2.4-.1.3-.2.4-.3.1-.6.1h-.3v.5l.4.1zm-.3-1.4v.7h.3c.2 0 .3 0 .4-.1.1-.1.1-.1.1-.3 0-.1 0-.2-.1-.3s-.2-.1-.4-.1c0 .1-.3.1-.3.1zm6.2 1.4h.2c.1 0 0 0 0 .1v.1h-1v-.2H72.2v-.5l-.5-.9h-.1v-.2h.6v.2H72l.4.7.4-.7h-.2v-.2h.5v.2H73l-.5.9v.5h.2zm1.3-.8c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.1.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.2-.2.3-.4.5-.5zm-.1.5c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1 0 .1-.1.2-.1.3zm0-.9c0 .1 0 .2.1.2s.2.1.3.1c.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1s-.2 0-.3.1c0 .1-.1.2-.1.3zm2 .4c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.1.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.1-.2.3-.4.5-.5zm-.1.5c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1-.1.1-.1.2-.1.3zm0-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1 0 .1-.1.2-.1.3zm1.9.2.2-.2c.1 0 .2-.1.3-.1.4 0 .7.3.7.7 0 .2-.1.4-.2.5s-.3.4-.5.4c-.1 0-.2 0-.3-.1-.1 0-.1-.1-.2-.2v.2c0 .1 0 0 0 0h-.4v-.2h.1v-1.6h-.1v-.2h.4v.8zm.4 1c.1 0 .3 0 .4-.1.1-.1.1-.2.1-.4 0-.3-.2-.5-.5-.5-.1 0-.3 0-.3.1-.1.1-.1.2-.1.4s0 .3.1.4.2.1.3.1zm3.4-.8c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.1.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4 0 .1-.2.2-.4.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.1-.2.2-.4.4-.5zm0 .5c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1-.1.1-.1.2-.1.3zm0-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1-.1.1-.1.2-.1.3zm1.9.4c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.1.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2s.2.2.2.4c0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4s-.2.2-.4.2-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.1-.2.2-.4.4-.5zm-.1.5c0 .1 0 .2.1.3s.2.1.3.1c.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1 0 .1-.1.2-.1.3zm.1-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1s-.2 0-.3.1c-.1.1-.1.2-.1.3zm1.9.4c-.1 0-.2-.1-.2-.2-.1 0-.1-.1-.1-.2 0-.2.1-.3.2-.4s.3-.2.4-.2c.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2v-.5c0-.2.1-.4.3-.5zm-.1.5c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1 0 .1-.1.2-.1.3zm0-.9c0 .1 0 .2.1.2s.2.1.3.1c.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1s-.2 0-.3.1c0 .1-.1.2-.1.3zm2 .4c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.1.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.2-.2.3-.4.5-.5zm-.1.5c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1-.1.1-.1.2-.1.3zm0-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1 0 .1-.1.2-.1.3zm1.9.2.2-.2c.1 0 .2-.1.3-.1.4 0 .7.3.7.7 0 .2-.1.4-.2.5s-.3.4-.5.4c-.1 0-.2 0-.3-.1-.1 0-.1-.1-.2-.2v.2c0 .1 0 0 0 0h-.4v-.2h.1v-1.6h-.1v-.2h.4v.8zm.4 1c.1 0 .3 0 .4-.1.1-.1.1-.2.1-.4 0-.3-.2-.5-.5-.5-.1 0-.3 0-.3.1-.1.2-.1.3-.1.4s0 .3.1.4.2.1.3.1zm7.2-.8c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4s.3-.2.4-.2c.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.2-.2.3-.4.5-.5zm-.1.5c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1s.1-.2.1-.3c0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1 0 .1-.1.2-.1.3zm0-.9c0 .1 0 .2.1.2s.2.1.3.1c.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1s-.2 0-.3.1c0 .1-.1.2-.1.3zm2 .4c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.1.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.2.2-.4.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.1-.2.2-.4.4-.5zm-.1.5c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1 0 .1-.1.2-.1.3zm0-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1 0 .1-.1.2-.1.3zm2 .4c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.1.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2s.2.2.2.4c0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4s-.3.2-.5.2-.3-.1-.5-.2-.2-.2-.2-.4c.1-.2.3-.4.5-.5zm-.1.5c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1-.1.1-.1.2-.1.3zm0-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1s-.2 0-.3.1c-.1.1-.1.2-.1.3zm3.8.4c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.1.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.2-.2.3-.4.5-.5zm0 .5c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1-.1.1-.1.2-.1.3zm0-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1-.1.1-.1.2-.1.3zm1.9.4c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.1.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2s.2.2.2.4c0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4s-.3.2-.5.2-.3-.1-.5-.2-.2-.2-.2-.4c.2-.2.3-.4.5-.5zm0 .5c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1-.1.1-.1.2-.1.3zm0-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1s-.2 0-.3.1c-.1.1-.1.2-.1.3zm1.9.4c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4s.3-.2.4-.2c.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.2-.2.3-.4.5-.5zm-.1.5c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1s.1-.2.1-.3c0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1 0 .1-.1.2-.1.3zm.1-.9c0 .1 0 .2.1.2s.2.1.3.1c.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1s-.2 0-.3.1c-.1.1-.1.2-.1.3zm11.3.4c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4s.3-.2.4-.2c.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.2-.2.3-.4.5-.5zm-.1.5c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1s.1-.2.1-.3c0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1 0 .1-.1.2-.1.3zm.1-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1s-.2 0-.3.1c-.1.1-.1.2-.1.3zm1.9.4c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.1.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.2-.2.3-.4.5-.5zm-.1.5c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1 0 .1-.1.2-.1.3zm0-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1 0 .1-.1.2-.1.3zm2 .4c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.1.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4s-.3.2-.5.2-.3-.1-.5-.2-.2-.2-.2-.4c.2-.2.3-.4.5-.5zm-.1.5c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1-.1.1-.1.2-.1.3zm0-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1s-.2 0-.3.1c0 .1-.1.2-.1.3zM2.5 47.3s.1 0 0 0l.4-.1-.4.8h-.1c-.1.1.1-.7.1-.7zm-.1-.7c0-.1 0-.1.1-.2s.1-.1.2-.1.2 0 .2.1.1.1.1.2 0 .1-.1.2-.1.1-.2.1-.2 0-.2-.1-.1-.1-.1-.2zm2.3.9h.6v.2H3.9v-.2h.6v-1h-.4v-.2h.6v1.2zm-.3-1.7v-.2h.2v.4c0 .1-.1 0-.1 0h-.1v-.2zm8.9 1.9h-.2v-.2H13.4v-1.4h-.2v-.2H14.7v.6h-.2v-.4h-.8v.5h.4v-.2h.2v.5h-.2v-.2h-.4v.7h.8V47h.2v.6h-.1l-1.3.1zm2.3-1c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.1.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.1-.2.3-.4.5-.5zm-.1.5c0 .1 0 .2.1.3s.2.1.3.1c.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1-.1.1-.1.2-.1.3zm0-.9c0 .1 0 .2.1.2s.2.1.3.1c.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1 0 .1-.1.2-.1.3zm2 .4c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4s.3-.2.4-.2c.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.1-.2.3-.4.5-.5zm-.1.5c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3s-.2-.1-.3-.1c-.1 0-.2 0-.3.1-.1.1-.1.2-.1.3zm0-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1s-.2 0-.3.1c-.1.1-.1.2-.1.3zm1.9.4c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4s.3-.2.4-.2c.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.2-.2.3-.4.5-.5zm0 .5c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1s.1-.2.1-.3c0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1-.1.1-.1.2-.1.3zm0-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1-.1.1-.1.2-.1.3zm2 1c.1-.1.1-.1 0 0 .1-.1.1-.1 0 0l.4-.1-.4.9h-.1c-.1 0 .1-.8.1-.8zm17 0c0-.1 0-.1 0 0 .1-.1.1-.1 0 0l.4-.1-.4.9h-.1c-.1 0 .1-.8.1-.8zm1.8-.6c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.1.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2s.2.2.2.4c0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.1-.2.3-.4.5-.5zm-.1.5c0 .1 0 .2.1.3s.2.1.3.1c.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3s-.2-.1-.3-.1c-.1 0-.2 0-.3.1-.1.1-.1.2-.1.3zm0-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1s-.2 0-.3.1c-.1.1-.1.2-.1.3zm2 .4c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.1.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.1-.2.3-.4.5-.5zm-.1.5c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1-.1.1-.1.2-.1.3zm0-.9c0 .1 0 .2.1.2s.2.1.3.1c.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1-.1.1-.1.2-.1.3zm1.9.4c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4s.3-.2.4-.2c.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.2-.2.3-.4.5-.5zm0 .5c0 .1 0 .2.1.3s.2.1.3.1.2 0 .3-.1.1-.2.1-.3c0-.1 0-.2-.1-.3s-.2-.1-.3-.1c-.1 0-.2 0-.3.1-.1.1-.1.2-.1.3zm0-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1s-.2.1-.3.1c-.1.1-.1.2-.1.3zm1.9.4c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.1.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2s.2.2.2.4c0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.2-.2.3-.4.5-.5zm0 .5c0 .1 0 .2.1.3s.2.1.3.1c.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3s-.2-.1-.3-.1c-.1 0-.2 0-.3.1-.1.1-.2.2-.1.3zm0-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1-.1.1-.1.2-.1.3zm2 1c0-.1 0-.1 0 0 .1-.1.1-.1 0 0l.4-.1-.4.9h-.1c-.1 0 .1-.8.1-.8zm13.1-.6c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4s.3-.2.4-.2c.2 0 .3.1.4.2s.2.2.2.4c0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.2-.2.3-.4.5-.5zm-.1.5c0 .1 0 .2.1.3s.2.1.3.1.2 0 .3-.1.1-.2.1-.3c0-.1 0-.2-.1-.3s-.2-.1-.3-.1c-.1 0-.2 0-.3.1 0 .1-.1.2-.1.3zm0-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1s-.2 0-.3.1c0 .1-.1.2-.1.3zm2 .4c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.1.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2s.2.2.2.4c0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.2-.2.3-.4.5-.5zm-.1.5c0 .1 0 .2.1.3s.2.1.3.1c.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1-.1.1-.1.2-.1.3zm0-.9c0 .1 0 .2.1.2s.2.1.3.1c.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1-.2 0-.4.1-.4.4 0-.1 0 0 0 0zm2 .4c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.1.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.1-.2.3-.4.5-.5zm-.1.5c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1-.1.1-.1.2-.1.3zm0-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1-.1.1-.1.2-.1.3zm9.5.4c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.1.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.2-.2.3-.4.5-.5zm-.1.5c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1 0 .1-.1.2-.1.3zm0-.9c0 .1 0 .2.1.2s.2.1.3.1c.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1s-.2.1-.3.1c0 .1-.1.2-.1.3zm2 .4c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.1.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.1-.2.3-.4.5-.5zm-.1.5c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3s-.2-.1-.3-.1c-.1 0-.2 0-.3.1-.1.1-.1.2-.1.3zm0-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1 0 .1-.1.2-.1.3zm2 .4c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4s.3-.2.4-.2c.2 0 .3.1.4.2s.2.2.2.4c0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.1-.2.3-.4.5-.5zm-.1.5c0 .1 0 .2.1.3s.2.1.3.1c.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1-.1.1-.1.2-.1.3zm0-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1s-.2 0-.3.1c-.1.1-.1.2-.1.3zm3.8.4c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.1.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.2-.2.3-.4.5-.5zm0 .5c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3s-.2-.1-.3-.1c-.1 0-.2 0-.3.1-.1.1-.1.2-.1.3zm0-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1-.3 0-.4.1-.4.4 0-.1 0 0 0 0zm1.9.4c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.1.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2s.2.2.2.4c0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.2-.2.3-.4.5-.5zm-.1.5c0 .1 0 .2.1.3s.2.1.3.1c.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1 0 .1-.1.2-.1.3zm.1-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1s-.2 0-.3.1c-.1.1-.1.2-.1.3zm1.9.4c-.1 0-.2-.1-.2-.2-.1-.1-.1-.1-.1-.2 0-.2.1-.3.2-.4s.3-.2.4-.2c.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2v-.5c0-.2.1-.4.3-.5zm-.1.5c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1 0 .1-.1.2-.1.3zm0-.9c0 .1 0 .2.1.2s.2.1.3.1c.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1s-.2 0-.3.1c0 .1-.1.2-.1.3zm2 .4c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.1.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.2-.2.3-.4.5-.5zm-.1.5c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3s-.2-.1-.3-.1c-.1 0-.2 0-.3.1-.1.1-.1.2-.1.3zm0-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1 0 .1-.1.2-.1.3zm2 .4c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.1.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2s.2.2.2.4c0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.1-.2.3-.4.5-.5zm-.1.5c0 .1 0 .2.1.3s.2.1.3.1c.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1-.1.1-.1.2-.1.3zm0-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1s-.2 0-.3.1c-.1.1-.1.2-.1.3zm1.9.2.2-.2c.1-.1.2-.1.3-.1.4 0 .7.3.7.7 0 .2-.1.4-.2.5-.1.1-.3.2-.5.2-.1 0-.2 0-.3-.1-.1 0-.1-.1-.2-.2v.2c0 .1 0 0 0 0h-.4v-.2h.1v-1.6h-.1v-.2h.4v1zm.4 1c.1 0 .3 0 .4-.1.1-.1.1-.2.1-.4 0-.3-.2-.5-.5-.5-.1 0-.3 0-.3.1-.1.1-.1.2-.1.4s0 .3.1.4c.1 0 .2.1.3.1zm5.3-.8c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4s.3-.2.4-.2c.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.2-.2.3-.4.5-.5zm-.1.5c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1s.1-.2.1-.3c0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1 0 .1-.1.2-.1.3zm0-.9c0 .1 0 .2.1.2s.2.1.3.1c.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1s-.2 0-.3.1c0 .1-.1.2-.1.3zm2 .4c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.1.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.2-.2.3-.4.5-.5zm-.1.5c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3s-.2-.1-.3-.1c-.1 0-.2 0-.3.1 0 .1-.1.2-.1.3zm0-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1 0 .1-.1.2-.1.3zm2 .4c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.1.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2s.2.2.2.4c0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.1-.2.3-.4.5-.5zm-.1.5c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1-.1.1-.1.2-.1.3zm0-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1s-.2 0-.3.1c-.1.1-.1.2-.1.3zm3.8.4c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.1.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.2-.2.3-.4.5-.5zm0 .5c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1-.1.1-.1.2-.1.3zm0-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1-.1.1-.1.2-.1.3zm1.9.4c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.1.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2s.2.2.2.4c0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.2-.2.3-.4.5-.5zm0 .5c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1-.1.1-.1.2-.1.3zm0-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1s-.2 0-.3.1c-.1.1-.1.2-.1.3zm1.9.4c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4s.3-.2.4-.2c.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.2-.2.3-.4.5-.5zm-.1.5c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1s.1-.2.1-.3c0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1 0 .1-.1.2-.1.3zm.1-.9c0 .1 0 .2.1.2s.2.1.3.1c.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1s-.3.1-.3.1c-.1.1-.1.2-.1.3zm11.3.4c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4s.3-.2.4-.2c.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.2-.2.3-.4.5-.5zm-.1.5c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1s.1-.2.1-.3c0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1 0 .1-.1.2-.1.3zm.1-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1s-.2 0-.3.1c-.1.1-.1.2-.1.3zm1.9.4c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.1.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.2-.2.3-.4.5-.5zm-.1.5c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1 0 .1-.1.2-.1.3zm0-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1-.2 0-.3.1-.4.4 0-.1 0 0 0 0zm2 .4c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.1.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.2-.2.3-.4.5-.5zm-.1.5c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1-.1.1-.1.2-.1.3zm0-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1s-.2 0-.3.1c0 .1-.1.2-.1.3zM13.6 51.5h-.4v-.2h.1v-1.4h-.1v-.2H14c.3 0 .5.1.6.2.2.2.2.4.2.7 0 .1 0 .3-.1.4s-.1.2-.2.3c-.1.1-.2.1-.3.2-.1 0-.3.1-.4.1h-.1c0-.1 0-.1-.1-.1zm0-.2h.2c.3 0 .4-.1.6-.2s.2-.3.2-.5-.1-.4-.2-.5c-.1-.1-.3-.2-.6-.2h-.2v1.4zm2-.8c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.1-.3.3-.4.5-.5zm-.1.4c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1-.1.2-.1.2-.1.3zm0-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1 0 .1-.1.2-.1.3zm2 .5c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.1-.3.3-.4.5-.5zm-.1.4c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1-.1.2-.1.2-.1.3zm0-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1s-.2 0-.3.1c-.1.1-.1.2-.1.3zm1.9.5c-.1 0-.2-.1-.2-.2s-.1-.2-.1-.3c0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.2-.3.3-.4.5-.5zm0 .4c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1V51c0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1v.2zm0-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1V50c0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1v.2zm2 1c.1 0 .1 0 0 0 .1 0 .1 0 0 0l.4-.1-.4.9h-.1c-.1 0 .1-.8.1-.8zm16.9-.6c0-.1 0-.1.1-.2s.1-.1.2-.1.2 0 .2.1.1.1.1.2 0 .1-.1.2-.1.1-.2.1-.2 0-.2-.1c-.1-.1-.1-.2-.1-.2zm0 .8c0-.1 0-.1.1-.2s.1-.1.2-.1.2 0 .2.1.1.1.1.2 0 .1-.1.2-.1.1-.2.1-.2 0-.2-.1-.1-.1-.1-.2zm1.9-.7c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.1-.3.3-.4.5-.5zm-.1.4c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1-.1.2-.1.2-.1.3zm0-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1s-.2 0-.3.1c-.1.1-.1.2-.1.3zm2 .5c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.1-.3.3-.4.5-.5zm-.1.4c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1-.1.2-.1.2-.1.3zm0-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1-.1.1-.1.2-.1.3zm1.9.5c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.2-.3.3-.4.5-.5zm0 .4c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1-.1.2-.1.2-.1.3zm0-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1s-.2 0-.3.1c-.1.1-.1.2-.1.3zm1.9.5c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.2-.3.3-.4.5-.5zm0 .4c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1-.1.2-.2.2-.1.3zm0-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1-.1.1-.1.2-.1.3zm1.9.4c0-.1 0-.1.1-.2s.1-.1.2-.1.2 0 .2.1.1.1.1.2 0 .1-.1.2-.1.1-.2.1-.2 0-.2-.1c-.1-.1-.1-.2-.1-.2zm0 .8c0-.1 0-.1.1-.2s.1-.1.2-.1.2 0 .2.1.1.1.1.2 0 .1-.1.2-.1.1-.2.1-.2 0-.2-.1c-.1 0-.1-.1-.1-.2zm13.2-.7c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.2-.3.3-.4.5-.5zm-.1.4c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1 0 .2-.1.2-.1.3zm0-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1s-.2 0-.3.1c0 .1-.1.2-.1.3zm2 .5c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.2-.3.3-.4.5-.5zm-.1.4c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1-.1.2-.1.2-.1.3zm0-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1-.2.1-.4.2-.4.4zm2 .5c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.1-.3.3-.4.5-.5zm-.1.4c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1-.1.2-.1.2-.1.3zm0-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1-.1.1-.1.2-.1.3zm17 .5c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.2-.3.3-.4.5-.5zm0 .4c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1-.1.2-.1.2-.1.3zm0-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1-.3.1-.4.2-.4.4zm1.9.5c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.2-.3.3-.4.5-.5zm-.1.4c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1 0 .2-.1.2-.1.3zm.1-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1s-.2 0-.3.1c-.1.1-.1.2-.1.3zm1.9.5c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2v-.5c0-.2.1-.3.3-.4zm-.1.4c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1 0-.1.1-.2.1-.2 0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1 0 .1-.1.1-.1.2zm0-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1s-.2 0-.3.1c0 .1-.1.2-.1.3zm2.6 1.3h.2c.1 0 0 0 0 .1v.1H87.1v-.2H87.4v-.5l-.5-.9h-.1v-.2h.6v.2h-.2l.4.7.4-.7h-.2v-.2h.5v.2h-.1l-.5.9v.5h.1zm1.3-.8c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.1-.3.3-.4.5-.5zm-.1.4c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1-.1.2-.1.2-.1.3zm0-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1s-.2 0-.3.1c-.1.1-.1.2-.1.3zm1.9.5c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.2-.3.4-.4.5-.5zm0 .4c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1-.1.2-.1.2-.1.3zm0-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1s-.2 0-.3.1c-.1.1-.1.2-.1.3zm1.8.3.2-.2c.1 0 .2-.1.3-.1.4 0 .7.3.7.7 0 .2-.1.4-.2.5s-.3.2-.5.2c-.1 0-.2 0-.3-.1-.1 0-.1-.1-.2-.2v.2c0 .1 0 0 0 0h-.4v-.2h.1v-1.6h-.1v-.2h.4v1zm.5 1c.1 0 .3 0 .4-.1.1-.1.1-.2.1-.4 0-.3-.2-.5-.5-.5-.1 0-.3 0-.3.1-.1.1-.1.2-.1.4 0 .1 0 .3.1.4.1 0 .2.1.3.1zm3.4-.8c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.2-.3.3-.4.5-.5zm-.1.4c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1 0 .2-.1.2-.1.3zm0-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1s-.2 0-.3.1c0 .1-.1.2-.1.3zm2 .5c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.2-.3.3-.4.5-.5zm-.1.4c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1 0 .2-.1.2-.1.3zm0-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1 0 .1-.1.2-.1.3zm2 .5c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.1-.3.3-.4.5-.5zm-.1.4c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1-.1.2-.1.2-.1.3zm0-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1s-.2 0-.3.1c-.1.1-.1.2-.1.3zm3.8.5c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.2-.3.3-.4.5-.5zm0 .4c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1-.1.2-.1.2-.1.3zm0-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1-.1.1-.1.2-.1.3zm1.9.5c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.2-.3.3-.4.5-.5zm0 .4c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1-.1.2-.1.2-.1.3zm0-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1s-.2 0-.3.1c-.1.1-.1.2-.1.3zm1.9.5c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.2-.3.3-.4.5-.5zm-.1.4c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1 0 .2-.1.2-.1.3zm.1-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1s-.2 0-.3.1c-.1.1-.1.2-.1.3zm11.3.5c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.2-.3.3-.4.5-.5zm-.1.4c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1 0 .2-.1.2-.1.3zm.1-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1s-.2 0-.3.1c-.1.1-.1.2-.1.3zm1.9.5c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.2-.3.3-.4.5-.5zm-.1.4c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1 0 .2-.1.2-.1.3zm0-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1-.2.1-.3.2-.4.4zm2 .5c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.2-.3.3-.4.5-.5zm-.1.4c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1-.1.2-.1.2-.1.3zm0-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1s-.2 0-.3.1c0 .1-.1.2-.1.3zM13.6 55.2h-.4V55h.1v-1.4h-.1v-.2H14c.3 0 .5.1.6.2.2.2.2.4.2.7 0 .1 0 .3-.1.4 0 .1-.1.2-.2.3-.1.1-.2.1-.3.2-.1 0-.3.1-.4.1h-.1c0-.1 0-.1-.1-.1zm0-.2h.2c.3 0 .4-.1.6-.2.1-.1.2-.3.2-.5s-.1-.4-.2-.5c-.1-.1-.3-.2-.6-.2h-.2V55zm2-.8c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.1-.2.3-.4.5-.5zm-.1.5c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1-.1.1-.1.2-.1.3zm0-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1 0 .1-.1.2-.1.3zm2 .4c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.1-.2.3-.4.5-.5zm-.1.5c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3s-.2-.1-.3-.1c-.1 0-.2 0-.3.1s-.1.2-.1.3zm0-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1s-.2 0-.3.1c-.1.1-.1.2-.1.3zm1.9.4c-.1 0-.2-.1-.2-.2 0 0-.1-.1-.1-.2 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c.1-.2 0-.3 0-.5s.1-.4.3-.5zm0 .5c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1-.1.1-.1.2-.1.3zm0-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1v-.2c0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1v.2zm2 1c.1 0 .1 0 0 0 .1-.1.1-.1 0 0l.4-.1-.4.9h-.1c-.1 0 .1-.8.1-.8zm16.9-.7c0-.1 0-.1.1-.2s.1-.1.2-.1.2 0 .2.1.1.1.1.2 0 .1-.1.2-.1.1-.2.1-.2 0-.2-.1-.1-.1-.1-.2zm0 .9c0-.1 0-.1.1-.2s.1-.1.2-.1.2 0 .2.1.1.1.1.2 0 .1-.1.2-.1.1-.2.1-.2 0-.2-.1-.1-.1-.1-.2zm1.9-.8c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.1-.2.3-.4.5-.5zm-.1.5c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3s-.2-.1-.3-.1c-.1 0-.2 0-.3.1s-.1.2-.1.3zm0-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1s-.2 0-.3.1c-.1.1-.1.2-.1.3zm2 .4c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.1-.2.3-.4.5-.5zm-.1.5c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1-.1.1-.1.2-.1.3zm0-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1-.1.1-.1.2-.1.3zm1.9.4c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.2-.2.3-.4.5-.5zm0 .5c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3s-.2-.1-.3-.1c-.1 0-.2 0-.3.1s-.1.2-.1.3zm0-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1s-.2 0-.3.1c-.1.1-.1.2-.1.3zm1.9.4c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.2-.2.3-.4.5-.5zm0 .5c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3s-.2-.1-.3-.1c-.1 0-.2 0-.3.1s-.1.2-.1.3zm0-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1-.1.1-.1.2-.1.3zm1.9.3c0-.1 0-.1.1-.2s.1-.1.2-.1.2 0 .2.1.1.1.1.2 0 .1-.1.2-.1.1-.2.1-.2 0-.2-.1c-.1 0-.1-.1-.1-.2zm0 .9c0-.1 0-.1.1-.2s.1-.1.2-.1.2 0 .2.1.1.1.1.2 0 .1-.1.2-.1.1-.2.1-.2 0-.2-.1c-.1-.1-.1-.1-.1-.2zm13.2-.8c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.2-.2.3-.4.5-.5zm-.1.5c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3s-.2-.1-.3-.1c-.1 0-.2 0-.3.1s-.1.2-.1.3zm0-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1s-.2 0-.3.1c0 .1-.1.2-.1.3zm2 .4c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.2-.2.3-.4.5-.5zm-.1.5c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1s-.1.2-.1.3zm0-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1-.2 0-.4.2-.4.4zm2 .4c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.1-.2.3-.4.5-.5zm-.1.5c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1-.1.1-.1.2-.1.3zm0-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1-.1.1-.1.2-.1.3zm5.7.4c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.2-.2.3-.4.5-.5zm0 .5c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1-.1.1-.2.2-.1.3zm0-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1-.1.1-.1.2-.1.3zm1.9.4c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.2-.2.3-.4.5-.5zm-.1.5c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1 0 .1-.1.2-.1.3zm0-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1s-.2 0-.3.1c0 .1-.1.2-.1.3zm2 .4c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.2-.2.3-.4.5-.5zm-.1.5c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1 0 .1-.1.2-.1.3zm0-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1s-.2 0-.3.1c0 .1-.1.2-.1.3zm2 .4c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.1-.2.3-.4.5-.5zm-.1.5c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3s-.2-.1-.3-.1c-.1 0-.2 0-.3.1-.1.1-.1.2-.1.3zm0-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1 0 .1-.1.2-.1.3zm2 .4c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.1-.2.3-.4.5-.5zm-.1.5c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1-.1.1-.1.2-.1.3zm0-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1s-.2 0-.3.1c-.1.1-.1.2-.1.3zm3.8.4c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.2-.2.3-.4.5-.5zm0 .5c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3s-.2-.1-.3-.1c-.1 0-.2 0-.3.1s-.1.2-.1.3zm0-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1-.3 0-.4.2-.4.4zm1.9.4c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.2-.2.3-.4.5-.5zm-.1.5c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1 0 .1-.1.2-.1.3zm.1-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1s-.2 0-.3.1c-.1.1-.1.2-.1.3zm1.9.4c-.1 0-.2-.1-.2-.2-.1 0-.1-.1-.1-.2 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2v-.5c0-.2.1-.4.3-.5zm-.1.5c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1 0 .1-.1.2-.1.3zm0-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1s-.2 0-.3.1c0 .1-.1.2-.1.3zm4.5 1.2h.2c.1 0 0 0 0 .1v.1h-1V55H89.2v-.5l-.5-.9h-.1v-.2h.6v.2H89l.4.7.4-.7h-.2v-.2h.5v.2H90l-.5.9v.5h.2zm1.2-.8c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4l.5-.5zm0 .5c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1-.1.1-.1.2-.1.3zm0-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1s-.2 0-.3.1c-.1.1-.1.2-.1.3zm1.9.4c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.2-.2.3-.4.5-.5zm0 .5c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3s-.2-.1-.3-.1c-.1 0-.2 0-.3.1s-.1.2-.1.3zm0-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1-.1.1-.1.2-.1.3zm1.8.2.2-.2c.1 0 .2-.1.3-.1.4 0 .7.3.7.7 0 .2-.1.4-.2.5-.1.1-.3.2-.5.2-.1 0-.2 0-.3-.1-.1 0-.1-.1-.2-.2v.2c0 .1 0 0 0 0h-.4v-.2h.1v-1.6h-.1V53h.4v1zm.5 1c.1 0 .3 0 .4-.1s.1-.2.1-.4c0-.3-.2-.5-.5-.5-.1 0-.3 0-.3.1s-.1.2-.1.4 0 .3.1.4.2.1.3.1zm1.5-.8c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.2-.2.3-.4.5-.5zm-.1.5c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1 0 .1-.1.2-.1.3zm0-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1s-.2 0-.3.1c0 .1-.1.2-.1.3zm2 .4c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.2-.2.3-.4.5-.5zm-.1.5c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3s-.2-.1-.3-.1c-.1 0-.2 0-.3.1s-.1.2-.1.3zm0-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1 0 .1-.1.2-.1.3zm2 .4c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.1-.2.3-.4.5-.5zm-.1.5c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1-.1.1-.1.2-.1.3zm0-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1s-.2 0-.3.1c-.1.1-.1.2-.1.3zm3.8.4c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.2-.2.3-.4.5-.5zm0 .5c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1s-.1.2-.1.3zm0-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1-.1.1-.1.2-.1.3zm1.9.4c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.2-.2.3-.4.5-.5zm0 .5c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1-.1.1-.1.2-.1.3zm0-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1s-.2 0-.3.1c-.1.1-.1.2-.1.3zm1.9.4c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.2-.2.3-.4.5-.5zm-.1.5c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1 0 .1-.1.2-.1.3zm.1-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1s-.2 0-.3.1c-.1.1-.1.2-.1.3zm11.3.4c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.2-.2.3-.4.5-.5zm-.1.5c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1 0 .1-.1.2-.1.3zm.1-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1s-.2 0-.3.1c-.1.1-.1.2-.1.3zm1.9.4c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.2-.2.3-.4.5-.5zm-.1.5c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1 0 .1-.1.2-.1.3zm0-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1-.2 0-.3.2-.4.4zm2 .4c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.2-.2.3-.4.5-.5zm-.1.5c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1-.1.1-.1.2-.1.3zm0-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1s-.2 0-.3.1c0 .1-.1.2-.1.3zM13.6 59h-.4v-.2h.1v-1.4h-.1v-.2H14c.3 0 .5.1.6.2.2.2.2.4.2.7 0 .1 0 .3-.1.4s-.1.2-.2.3c-.1.1-.2.1-.3.2h-.6zm0-.2h.2c.3 0 .4-.1.6-.2.1-.1.2-.3.2-.5s-.1-.4-.2-.5c-.1-.1-.3-.2-.6-.2h-.2v1.4zm2-.8c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.1-.2.3-.4.5-.5zm-.1.5c0 .1 0 .2.1.3s.2.1.3.1c.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1-.1.1-.1.2-.1.3zm0-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1 0 .1-.1.2-.1.3zm2 .4c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.1-.2.3-.4.5-.5zm-.1.5c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3s-.2-.1-.3-.1c-.1 0-.2 0-.3.1-.1.1-.1.2-.1.3zm0-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1s-.2 0-.3.1c-.1.1-.1.2-.1.3zm1.9.4c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3 0 .1-.1.2-.2.2.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.2-.2.3-.4.5-.5zm0 .5c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1s.1-.2.1-.3c0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1-.1.1-.1.2-.1.3zm0-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1-.1.1-.1.2-.1.3zm2 1c.1-.1.1-.1 0 0 .1-.1.1-.1 0 0l.4-.1-.4.9h-.1c-.1 0 .1-.8.1-.8zm16.9-.7c0-.1 0-.1.1-.2s.1-.1.2-.1.2 0 .2.1.1.1.1.2 0 .1-.1.2-.1.1-.2.1-.2 0-.2-.1-.1-.1-.1-.2zm0 .9c0-.1 0-.1.1-.2s.1-.1.2-.1.2 0 .2.1.1.1.1.2 0 .1-.1.2h-.2c-.1 0-.2 0-.2-.1s-.1-.1-.1-.1zm1.9-.8c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.1-.2.3-.4.5-.5zm-.1.5c0 .1 0 .2.1.3s.2.1.3.1c.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3s-.2-.1-.3-.1c-.1 0-.2 0-.3.1-.1.1-.1.2-.1.3zm0-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1s-.2 0-.3.1c-.1.1-.1.2-.1.3zm2 .4c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.1-.2.3-.4.5-.5zm-.1.5c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1-.1.1-.1.2-.1.3zm0-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1-.1.1-.1.2-.1.3zm1.9.4c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.2-.2.3-.4.5-.5zm0 .5c0 .1 0 .2.1.3s.2.1.3.1.2 0 .3-.1.1-.2.1-.3c0-.1 0-.2-.1-.3s-.2-.1-.3-.1c-.1 0-.2 0-.3.1-.1.1-.1.2-.1.3zm0-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1s-.2 0-.3.1c-.1.1-.1.2-.1.3zm1.9.4c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.2-.2.3-.4.5-.5zm0 .5c0 .1 0 .2.1.3s.2.1.3.1c.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3s-.2-.1-.3-.1c-.1 0-.2 0-.3.1-.1.1-.1.2-.1.3zm0-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1-.1.1-.1.2-.1.3zm1.9.3c0-.1 0-.1.1-.2s.1-.1.2-.1.2 0 .2.1.1.1.1.2 0 .1-.1.2-.1.1-.2.1-.2 0-.2-.1c-.1 0-.1-.1-.1-.2zm0 .9c0-.1 0-.1.1-.2s.1-.1.2-.1.2 0 .2.1.1.1.1.2 0 .1-.1.2h-.2c-.1 0-.2 0-.2-.1-.1 0-.1-.1-.1-.1zm13.2-.8c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.2-.2.3-.4.5-.5zm-.1.5c0 .1 0 .2.1.3s.2.1.3.1.2 0 .3-.1.1-.2.1-.3c0-.1 0-.2-.1-.3s-.2-.1-.3-.1c-.1 0-.2 0-.3.1 0 .1-.1.2-.1.3zm0-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1s-.2 0-.3.1c0 .1-.1.2-.1.3zm2 .4c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.2-.2.3-.4.5-.5zm-.1.5c0 .1 0 .2.1.3s.2.1.3.1c.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1-.1.1-.1.2-.1.3zm0-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1 0 .1-.1.2-.1.3zm2 .4c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.1-.2.3-.4.5-.5zm-.1.5c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1-.1.1-.1.2-.1.3zm0-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1-.1.1-.1.2-.1.3zm9.5.4c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.2-.1-.4-.2c-.1-.1-.2-.3-.2-.4.1-.2.2-.4.4-.5zm-.1.5c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1 0 .1-.1.2-.1.3zm0-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1s-.2 0-.3.1c0 .1-.1.2-.1.3zm2 .4c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.1-.2.3-.4.5-.5zm-.1.5c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3s-.2-.1-.3-.1c-.1 0-.2 0-.3.1-.1.1-.1.2-.1.3zm0-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1 0 .1-.1.2-.1.3zm2 .4c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.1-.2.3-.4.5-.5zm-.1.5c0 .1 0 .2.1.3s.2.1.3.1c.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1-.1.1-.1.2-.1.3zm0-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1s-.2 0-.3.1c-.1.1-.1.2-.1.3zm3.8.4c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.2-.2.3-.4.5-.5zm0 .5c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3s-.2-.1-.3-.1c-.1 0-.2 0-.3.1-.1.1-.1.2-.1.3zm0-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1-.1.1-.1.2-.1.3zm1.9.4c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3 0 .1-.1.2-.2.2.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.2-.2.3-.4.5-.5zm-.1.5c0 .1 0 .2.1.3s.2.1.3.1c.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1 0 .1-.1.2-.1.3zm.1-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1s-.2 0-.3.1c-.1.1-.1.2-.1.3zm1.9.4c-.1 0-.2-.1-.2-.2-.1 0-.1-.1-.1-.2 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3 0 0-.1.1-.2.1.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2v-.4c0-.2.1-.4.3-.5zm-.1.5c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1 0 .1-.1.2-.1.3zm0-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1s-.2 0-.3.1c0 .1-.1.2-.1.3zm6.4 1.2h.2c.1 0 0 0 0 .1v.1H90.9v-.2H91.2v-.5l-.5-.9h-.1v-.2h.5v.2h-.2l.4.7.4-.7h-.2v-.2h.5v.2h-.1l-.5.9v.5h.2zm1.2-.8c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.2-.2.3-.4.5-.5zm0 .5c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3s-.2-.1-.3-.1c-.1 0-.2 0-.3.1-.1.1-.1.2-.1.3zm0-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1-.1.1-.1.2-.1.3zm1.9.4c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.2-.2.3-.4.5-.5zm-.1.5c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1 0 .1-.1.2-.1.3zm.1-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1s-.2 0-.3.1c-.1.1-.1.2-.1.3zm1.9.4c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.2-.2.3-.4.5-.5zm-.1.5c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1s.1-.2.1-.3c0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1 0 .1-.1.2-.1.3zm0-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1s-.2 0-.3.1c0 .1-.1.2-.1.3zm2 .4c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.2-.2.3-.4.5-.5zm-.1.5c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3s-.2-.1-.3-.1c-.1 0-.2 0-.3.1 0 .1-.1.2-.1.3zm0-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1 0 .1-.1.2-.1.3zm2 .4c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.1-.2.3-.4.5-.5zm-.1.5c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1-.1.1-.1.2-.1.3zm0-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1s-.2 0-.3.1c-.1.1-.1.2-.1.3zm3.8.4c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.2-.2.3-.4.5-.5zm0 .5c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1-.1.1-.1.2-.1.3zm0-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1-.1.1-.1.2-.1.3zm1.9.4c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.2-.2.3-.4.5-.5zm0 .5c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1-.1.1-.1.2-.1.3zm0-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1s-.2 0-.3.1c-.1.1-.1.2-.1.3zm1.9.4c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.2-.2.3-.4.5-.5zm-.1.5c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1s.1-.2.1-.3c0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1 0 .1-.1.2-.1.3zm.1-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1s-.2 0-.3.1c-.1.1-.1.2-.1.3zm11.3.4c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.2-.2.3-.4.5-.5zm-.1.5c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1s.1-.2.1-.3c0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1 0 .1-.1.2-.1.3zm.1-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1s-.2 0-.3.1c-.1.1-.1.2-.1.3zm1.9.4c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.2-.2.3-.4.5-.5zm-.1.5c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1 0 .1-.1.2-.1.3zm0-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1 0 .1-.1.2-.1.3zm2 .4c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.2-.2.3-.4.5-.5zm-.1.5c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1-.1.1-.1.2-.1.3zm0-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1s-.2 0-.3.1c0 .1-.1.2-.1.3zM13.6 62.8h-.4v-.2h.1v-1.4h-.1V61H14c.3 0 .5.1.6.2.2.2.2.4.2.7 0 .1 0 .3-.1.4s-.1.2-.2.3c-.1.1-.2.1-.3.2-.1 0-.3.1-.4.1h-.1c0-.1 0-.1-.1-.1zm0-.2h.2c.3 0 .4-.1.6-.2s.2-.3.2-.5-.1-.4-.2-.5c-.1-.1-.3-.2-.6-.2h-.2v1.4zm2-.8c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.1-.3.3-.4.5-.5zm-.1.4c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1-.1.2-.1.3-.1.3zm0-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2h-.3c-.1 0-.2 0-.3.1 0 .1-.1.1-.1.2zm2 .5c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.1-.3.3-.4.5-.5zm-.1.4c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1-.1.2-.1.3-.1.3zm0-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2h-.3c-.1 0-.2 0-.3.1-.1.1-.1.1-.1.2zm1.9.5c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.2-.3.3-.4.5-.5zm0 .4c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1s.1-.2.1-.3c0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1-.1.2-.1.3-.1.3zm0-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2h-.4c-.1 0-.2 0-.3.1v.2zm2 1c.1 0 .1 0 0 0 .1 0 .1 0 0 0l.4-.1-.4.9h-.1c-.1 0 .1-.8.1-.8zm16.9-.6c0-.1 0-.1.1-.2s.1-.1.2-.1.2 0 .2.1.1.1.1.2 0 .1-.1.2-.1.1-.2.1-.2 0-.2-.1c-.1-.1-.1-.1-.1-.2zm0 .8c0-.1 0-.1.1-.2s.1-.1.2-.1.2 0 .2.1.1.1.1.2 0 .1-.1.2-.1.1-.2.1-.2 0-.2-.1-.1-.1-.1-.2zm1.9-.7c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.1-.3.3-.4.5-.5zm-.1.4c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1-.1.2-.1.3-.1.3zm0-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2h-.3c-.1 0-.2 0-.3.1-.1.1-.1.1-.1.2zm2 .5c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.1-.3.3-.4.5-.5zm-.1.4c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1-.1.2-.1.3-.1.3zm0-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2h-.3c-.1 0-.2 0-.3.1-.1.1-.1.1-.1.2zm1.9.5c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.2-.3.3-.4.5-.5zm0 .4c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1s.1-.2.1-.3c0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1-.1.2-.1.3-.1.3zm0-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2h-.4c-.1 0-.2 0-.3.1v.2zm1.9.5c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.2-.3.3-.4.5-.5zm0 .4c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1-.1.2-.1.3-.1.3zm0-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2H46c-.1 0-.2 0-.3.1v.2zm1.9.4c0-.1 0-.1.1-.2s.1-.1.2-.1.2 0 .2.1.1.1.1.2 0 .1-.1.2c0 0-.1.1-.2.1s-.2 0-.2-.1c-.1-.1-.1-.1-.1-.2zm0 .8c0-.1 0-.1.1-.2s.1-.1.2-.1.2 0 .2.1.1.1.1.2 0 .1-.1.2-.1.1-.2.1-.2 0-.2-.1c-.1 0-.1-.1-.1-.2zm13.8.1h.2c.1 0 0 0 0 .1v.1H60.7v-.2H61v-.5l-.5-.9h-.1V61h.6v.2h-.2l.4.7.4-.7h-.2V61h.5v.2h-.1l-.5.9v.5h.1zm1.3-.8c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.2-.3.3-.4.5-.5zm-.1.4c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1-.1.2-.1.3-.1.3zm0-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2H63c-.1 0-.2 0-.3.1 0 .1-.1.1-.1.2zm2 .5c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.1-.3.3-.4.5-.5zm-.1.4c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1-.1.2-.1.3-.1.3zm0-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2h-.3c-.1 0-.2 0-.3.1-.1.1-.1.1-.1.2zm1.9.3.2-.2c.1 0 .2-.1.3-.1.4 0 .7.3.7.7 0 .2-.1.4-.2.5s-.3.2-.5.2c-.1 0-.2 0-.3-.1-.1 0-.1-.1-.2-.2v.2c0 .1 0 0 0 0h-.5v-.2h.1v-1.6h.4v.8zm.4 1c.1 0 .3 0 .4-.1s.1-.2.1-.4c0-.3-.2-.5-.5-.5-.1 0-.3 0-.3.1s-.1.2-.1.4 0 .3.1.4.2.1.3.1zm6.1.2c-.1 0-.1 0 0 0-.1 0-.1-.1 0 0l-.1-.2-.2.2c-.1 0-.2.1-.3.1-.2 0-.4-.1-.5-.2-.1-.1-.2-.3-.2-.5 0-.4.3-.7.7-.7.1 0 .2 0 .3.1.1 0 .2.1.2.2V61H72.5v-.2h.6v1.7h.2v.2h-.1l-.3.1zm-1-.7c0 .1 0 .3.1.3s.2.1.3.1.3 0 .3-.1.1-.2.1-.4c0-.3-.2-.5-.4-.5-.1 0-.3 0-.3.1-.1.2-.1.4-.1.5zm2.1-.3c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.2-.3.3-.4.5-.5zm-.1.4c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1 0 .2-.1.3-.1.3zm0-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2h-.3c-.1 0-.2 0-.3.1 0 .1-.1.1-.1.2zm2 .5c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.1-.3.3-.4.5-.5zm-.1.4c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1-.1.2-.1.3-.1.3zm0-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2h-.3c-.1 0-.2 0-.3.1 0 .1-.1.1-.1.2zm2.3 1.3h.1v.2h-1v-.2H77.5v-1.4H77.2V61H78c.2 0 .4 0 .6.1s.2.2.2.4-.1.3-.2.4-.3.1-.6.1h-.3v.5l.4.1zm-.3-1.4v.7h.3c.2 0 .3 0 .4-.1.1-.1.1-.1.1-.3 0-.1 0-.2-.1-.3s-.2-.1-.4-.1c0 .1-.3.1-.3.1zm3.7.6c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.2-.3.3-.4.5-.5zm0 .4c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1-.1.2-.1.3-.1.3zm0-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2h-.4c-.1 0-.2 0-.3.1v.2zm1.9.5c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.2-.3.3-.4.5-.5zm-.1.4c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1 0 .2-.1.3-.1.3zm.1-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2h-.4c-.1 0-.2 0-.3.1v.2zm1.9.5c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2v-.5c0-.2.1-.3.3-.4zm-.1.4c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1 0 .2-.1.3-.1.3zm0-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2h-.3c-.1 0-.2 0-.3.1 0 .1-.1.1-.1.2zm8.3 1.3h.2c.1 0 0 0 0 .1v.1H92.8v-.2h.2v-.5l-.5-.9h-.1V61h.6v.2h-.2l.4.7.4-.7h-.2V61h.5v.2h-.1l-.5.9v.5h.2zm1.2-.8c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.2-.3.3-.4.5-.5zm-.1.4c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1 0 .2-.1.3-.1.3zm.1-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2H95c-.1 0-.2 0-.3.1v.2zm1.9.5c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.2-.3.3-.4.5-.5zm-.1.4c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1s.1-.2.1-.3c0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1 0 .2-.1.3-.1.3zm0-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2h-.3c-.1 0-.2 0-.3.1 0 .1-.1.1-.1.2zm2 .5c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.2-.3.3-.4.5-.5zm-.1.4c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1 0 .2-.1.3-.1.3zm0-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2h-.3c-.1 0-.2 0-.3.1 0 .1-.1.1-.1.2zm2 .5c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.1-.3.3-.4.5-.5zm-.1.4c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1-.1.2-.1.3-.1.3zm0-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1s-.2 0-.3.1c-.1.2-.1.2-.1.3zm4.5 1.3h.2c.1 0 0 0 0 .1v.1H104.1v-.2H104.4v-.5l-.5-.9h-.1V61h.6v.2h-.2l.4.7.4-.7h-.2V61h.5v.2h-.1l-.5.9v.5h.1zm1.2-.8c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.2-.3.3-.4.5-.5zm0 .4c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1-.1.2-.1.3-.1.3zm0-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1s-.2 0-.3.1c-.1.2-.1.2-.1.3zm1.9.5c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.2-.3.3-.4.5-.5zm-.1.4c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1s.1-.2.1-.3c0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1 0 .2-.1.3-.1.3zm.1-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1s-.2 0-.3.1c-.1.2-.1.2-.1.3zm1.8.3.2-.2c.1 0 .2-.1.3-.1.4 0 .7.3.7.7 0 .2-.1.4-.2.5s-.3.2-.5.2c-.1 0-.2 0-.3-.1-.1 0-.1-.1-.2-.2v.2c0 .1 0 0 0 0h-.4v-.2h.1V61h-.1v-.2h.4v.8zm.5 1c.1 0 .3 0 .4-.1.1-.1.1-.2.1-.4 0-.3-.2-.5-.5-.5-.1 0-.3 0-.3.1-.1.1-.1.2-.1.4s0 .3.1.4c0 0 .1.1.3.1zm1.5-.1c0-.1 0-.1.1-.2s.1-.1.2-.1.2 0 .2.1.1.1.1.2 0 .1-.1.2-.1.1-.2.1-.2 0-.2-.1-.1-.1-.1-.2zm3.7 0c0-.1 0-.1.1-.2s.1-.1.2-.1.2 0 .2.1.1.1.1.2 0 .1-.1.2-.1.1-.2.1-.2 0-.2-.1-.1-.1-.1-.2zm2.7.3c-.1 0-.1-.1 0 0l-.1-.2-.2.2c-.1 0-.2.1-.3.1-.2 0-.4-.1-.5-.2-.1-.1-.2-.3-.2-.5 0-.4.3-.7.7-.7.1 0 .2 0 .3.1.1 0 .2.1.2.2V61h-.2v-.2h.5v1.7h.2v.2h-.1l-.3.1zm-1-.7c0 .1 0 .3.1.3s.2.1.3.1.3 0 .3-.1.1-.2.1-.4c0-.3-.2-.5-.5-.5-.1 0-.3 0-.3.1v.5zm2.1-.3c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.2-.3.3-.4.5-.5zm-.1.4c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1s.1-.2.1-.3c0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1 0 .2-.1.3-.1.3zm.1-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1s-.2 0-.3.1c-.1.2-.1.2-.1.3zm1.9.5c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.2-.3.3-.4.5-.5zm-.1.4c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1 0 .2-.1.3-.1.3zm0-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1 0 .2-.1.2-.1.3zm2.3 1.3h.1v.2h-1v-.2H122.7v-1.4H122.4V61H123.2c.2 0 .4 0 .6.1s.2.2.2.4-.1.3-.2.4-.3.1-.6.1h-.2v.5l.3.1zm-.3-1.4v.7h.3c.2 0 .3 0 .4-.1.1-.1.1-.1.1-.3 0-.1 0-.2-.1-.3s-.2-.1-.4-.1c0 .1-.3.1-.3.1zM13.7 65.6c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.2-.3.3-.5.5-.5zm-.1.4c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1 0 .1-.1.2-.1.3zm0-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1s-.2 0-.3.1c0 .1-.1.2-.1.3zm2 .5c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.1-.3.3-.5.5-.5zm-.1.4c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1-.1.1-.1.2-.1.3zm0-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1 0 .1-.1.2-.1.3zm2 .5c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.1-.3.3-.5.5-.5zm-.1.4c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1-.1.1-.1.2-.1.3zm0-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1s-.2 0-.3.1c-.1.1-.1.2-.1.3zm1.8 1.1.3-.9v-.1H19.8v.1l.3.9.1-1.2H19.9v-.2h.7v.2h-.1l-.2 1.5v.1h-.2c-.1 0 0 0 0-.1l-.3-1.1-.3 1.1v.1c0 .1 0 0-.1 0h-.1v-.1l-.2-1.5H19v-.2h.7v.2H19.4c-.3-.1-.2 1.2-.2 1.2zm2.1-.1c.1 0 .1 0 0 0 .1-.1.1-.1 0 0l.4-.1-.3 1h-.1c-.2-.1 0-.9 0-.9zm16.9-.7c0-.1 0-.1.1-.2s.1-.1.2-.1.2 0 .2.1c.1.1.1.1.1.2s0 .1-.1.2-.1.1-.2.1-.2 0-.2-.1-.1-.1-.1-.2zm0 .9c0-.1 0-.1.1-.2s.1-.1.2-.1.2 0 .2.1.1.1.1.2 0 .1-.1.2-.1.1-.2.1-.2 0-.2-.1-.1-.1-.1-.2zm1.9-.7c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1 0 .1.1.1.2 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.1-.2.3-.4.5-.4zm-.1.4c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1-.1.1-.1.2-.1.3zm0-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1s-.2 0-.3.1c-.1.1-.1.2-.1.3zm2 .5c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.1-.3.3-.5.5-.5zm-.1.4c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1-.1.1-.1.2-.1.3zm0-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1-.1.1-.1.2-.1.3zm1.9.5c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.2-.3.3-.5.5-.5zm0 .4c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1-.1.1-.1.2-.1.3zm0-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1s-.2 0-.3.1c-.1.1-.1.2-.1.3zm1.9.5c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.2-.3.3-.5.5-.5zm0 .4c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1-.1.1-.1.2-.1.3zm0-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1-.1.1-.1.2-.1.3zm1.9.3c0-.1 0-.1.1-.2s.1-.1.2-.1.2 0 .2.1c.1.1.1.1.1.2s0 .1-.1.2-.1.1-.2.1-.2 0-.2-.1c-.1 0-.1-.1-.1-.2zm0 .9c0-.1 0-.1.1-.2s.1-.1.2-.1.2 0 .2.1c.1.1.1.1.1.2s0 .1-.1.2-.1.1-.2.1-.2 0-.2-.1c-.1-.1-.1-.1-.1-.2zm15.1-1.1c-.1 0-.1-.1 0 0l-.1-.7v-.1H62.9v.7c-.1 0-.1.1-.2.1.1 0 .1 0 0 0zm.5 0s0-.1 0 0l-.1-.7v-.1H63.4v.7c-.1 0-.1.1-.2.1.1 0 .1 0 0 0zm2 1.1h.2c.1 0 0 0 0 .1v.1H64.5v-.2H64.8v-.5l-.5-.9h-.1v-.2h.6v.2h-.1l.4.7.4-.7h-.2v-.2h.5v.2h-.1l-.7.9v.5h.2zm1.2-.7c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.2-.3.4-.5.5-.5zm0 .4c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1-.1.1-.1.2-.1.3zm0-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1s-.2 0-.3.1c-.1.1-.1.2-.1.3zm1.9.5c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c.1-.2 0-.3 0-.5s.1-.4.3-.4zm0 .4c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1V66c0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1v.3zm0-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1s-.2 0-.3.1c-.1.1-.1.2-.1.3zm1.9.5c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.2-.3.3-.5.5-.5zm0 .4c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1-.1.1-.2.2-.1.3zm0-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1-.1.1-.1.2-.1.3zm1.9.5c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1 0 .1.1.1.2 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.2-.2.3-.4.5-.4zm-.1.4c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1 0 .1-.1.2-.1.3zm0-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1s-.2 0-.3.1c0 .1-.1.2-.1.3zm2.4 1.2h.1v.2h-1v-.2H73.8v-1.4H73.5v-.2H74.3c.2 0 .4 0 .6.1.1.1.2.2.2.4s-.1.3-.2.4c-.1.1-.3.1-.6.1H74v.5l.4.1zm-.4-1.4v.7h.3c.2 0 .3 0 .4-.1.1-.1.1-.1.1-.3 0-.1 0-.2-.1-.3s-.2-.1-.4-.1c0 .1-.3.1-.3.1zm1.9.7c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.1-.3.3-.5.5-.5zm-.1.4c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1-.1.1-.1.2-.1.3zm0-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1 0 .1-.1.2-.1.3zm2 .5c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.1-.3.3-.5.5-.5zm-.1.4c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1-.1.1-.1.2-.1.3zm0-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1s-.2 0-.3.1c-.1.1-.1.2-.1.3zm3.8.5c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.2-.3.3-.5.5-.5zm0 .4c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1-.1.1-.1.2-.1.3zm0-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1-.1.1-.1.2-.1.3zm1.9.5c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.2-.3.3-.5.5-.5zm-.1.4c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1 0 .1-.1.2-.1.3zm.1-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1s-.2 0-.3.1c-.1.1-.1.2-.1.3zm1.9.5c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2V66c0-.2.1-.4.3-.4zm-.1.4c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1 0 .1-.1.2-.1.3zm0-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1s-.2 0-.3.1c0 .1-.1.2-.1.3zm10.2 1.2h.2c.1 0 0 0 0 .1v.1H94.7v-.2H95v-.5l-.5-.9h-.1v-.2h.6v.2h-.1l.4.7.4-.7h-.2v-.2h.5v.2h-.1l-.5.9v.5zm1.2-.7c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.2-.3.3-.5.5-.5zm-.1.4c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1 0 .1-.1.2-.1.3zm0-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1s-.2 0-.3.1c0 .1-.1.2-.1.3zm2 .5c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.2-.3.3-.5.5-.5zm-.1.4c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1 0 .1-.1.2-.1.3zm0-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1 0 .1-.1.2-.1.3zm2 .5c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.1-.3.3-.5.5-.5zm-.1.4c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1-.1.1-.1.2-.1.3zm0-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1s-.2 0-.3.1c-.1.1-.1.2-.1.3zm5.7.1s0-.1 0 0l-.1-.7v-.1H106.2c.1 0 0 0 0 .1v.6c-.1 0-.1.1-.2.1.1 0 .1 0 0 0zm.6 0s-.1 0 0 0c-.1 0-.1-.1 0 0l-.1-.7v-.1H106.8v.7c-.1 0-.1.1-.2.1.1 0 0 0 0 0zm1.9 1.1h.2c.1 0 0 0 0 .1v.1H107.8v-.2H108.1v-.5l-.5-.9h-.1v-.2h.6v.2h-.2l.4.7.4-.7h-.2v-.2h.5v.2h-.1l-.5.9v.5h.1zm1.3-.7c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.2-.3.3-.5.5-.5zm-.1.4c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1 0 .1-.1.2-.1.3zm0-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1 0 .1-.1.2-.1.3zm2 .5c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.2-.3.3-.5.5-.5zm-.1.4c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1-.1.1-.1.2-.1.3zm0-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1s-.2 0-.3.1c0 .1-.1.2-.1.3zm2 .5c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.1-.3.3-.5.5-.5zm-.1.4c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1-.1.1-.1.2-.1.3zm0-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1s-.2 0-.3.1c-.1.1-.1.2-.1.3zm1.9.5c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.2-.3.3-.5.5-.5zm0 .4c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1-.1.1-.1.2-.1.3zm0-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1-.1.1-.1.2-.1.3zm1.9.5c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.2-.3.3-.5.5-.5zm0 .4c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1-.1.1-.1.2-.1.3zm0-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1s-.2 0-.3.1c-.1.1-.1.2-.1.3zm2.3 1.2h.1v.2h-1v-.2H119v-1.4H118.7v-.2H119.5c.2 0 .4 0 .6.1.1.1.2.2.2.4s-.1.3-.2.4c-.1.1-.3.1-.6.1h-.3v.5l.4.1zm-.3-1.4v.7h.3c.2 0 .3 0 .4-.1s.1-.1.1-.3c0-.1 0-.2-.1-.3s-.2-.1-.4-.1c-.1.1-.3.1-.3.1zm1.8.3c-.1 0-.1-.1 0 0l-.1-.7v-.1H121.3c.1 0 0 0 0 .1v.6c-.1 0-.1.1-.2.1.1 0 .1 0 0 0zm.6 0c-.1 0-.1 0 0 0-.1 0-.1-.1 0 0l-.1-.7v-.1H121.9v.7c-.1 0-.1.1-.2.1zM13.6 69.9l.3-.9v-.1h.2v.1l.3.9.1-1.2H14.2v-.2h.7v.2h-.1l-.2 1.5v.1h-.2v-.1l-.4-1-.3 1.1v.1h-.2v-.1l-.2-1.5h-.1v-.2h.7v.2H13.6c-.2-.1 0 1.1 0 1.1zm2-.6c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.1-.2.3-.4.5-.5zm-.1.5c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1-.1.1-.1.2-.1.3zm0-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1 0 .1-.1.2-.1.3zm2 .4c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.1-.2.3-.4.5-.5zm-.1.5c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1-.1.1-.1.2-.1.3zm0-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1s-.2 0-.3.1c-.1.1-.1.2-.1.3zm1.8 1 .3-.9v-.1H19.8v.1l.3.9.1-1.2H19.9v-.2h.7v.2h-.1l-.2 1.5v.1h-.2v-.1l-.3-1.1-.3 1.1v.1c0 .1 0 0-.1 0h-.1v-.1l-.2-1.5H19v-.2h.7v.2H19.4c-.3 0-.2 1.2-.2 1.2zm2.1 0c.1-.1.1-.1 0 0 .1-.1.1-.1 0 0l.4-.1-.4.9h-.1c-.1 0 .1-.8.1-.8zm16.9-.7c0-.1 0-.1.1-.2s.1-.1.2-.1.2 0 .2.1.1.1.1.2 0 .1-.1.2-.1.1-.2.1-.2 0-.2-.1-.1-.1-.1-.2zm0 .9c0-.1 0-.1.1-.2s.1-.1.2-.1.2 0 .2.1.1.1.1.2 0 .1-.1.2-.1.1-.2.1-.2 0-.2-.1-.1-.2-.1-.2zm1.9-.8c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.1-.2.3-.4.5-.5zm-.1.5c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1-.1.1-.1.2-.1.3zm0-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1s-.2 0-.3.1c-.1.1-.1.2-.1.3zm2 .4c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.1-.2.3-.4.5-.5zm-.1.5c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1-.1.1-.1.2-.1.3zm0-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1-.1.1-.1.2-.1.3zm1.9.4c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.2-.2.3-.4.5-.5zm0 .5c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1-.1.1-.1.2-.1.3zm0-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1s-.2 0-.3.1c-.1.1-.1.2-.1.3zm1.9.4c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.2-.2.3-.4.5-.5zm0 .5c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1-.1.1-.1.2-.1.3zm0-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1-.1.1-.1.2-.1.3zm1.9.3c0-.1 0-.1.1-.2s.1-.1.2-.1.2 0 .2.1c.1.1.1.1.1.2s0 .1-.1.2-.1.1-.2.1-.2 0-.2-.1c-.1 0-.1-.1-.1-.2zm0 .9c0-.1 0-.1.1-.2s.1-.1.2-.1.2 0 .2.1c.1.1.1.1.1.2s0 .1-.1.2-.1.1-.2.1-.2 0-.2-.1c-.1-.1-.1-.2-.1-.2zm-34 3.6.3-.9v-.1h.2v.1l.3.9.1-1.2H14.2v-.2h.7v.2h-.1l-.2 1.5v.1h-.2c-.1 0 0 0 0-.1l-.4-1-.3 1v.1h-.2V74l-.2-1.5h-.1v-.2h.7v.2H13.6c-.2 0 0 1.2 0 1.2zm2-.6c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.1-.3.3-.4.5-.5zm-.1.5c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1-.1.1-.1.2-.1.3zm0-1c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1 0 .2-.1.2-.1.3zm2 .5c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.1-.3.3-.4.5-.5zm-.1.5c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1-.1.1-.1.2-.1.3zm0-1c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1s-.2 0-.3.1c-.1.2-.1.2-.1.3zm1.8 1.1.3-.9v-.1H19.8v.1l.3.9.1-1.2H19.9v-.2h.7v.2h-.1l-.3 1.5v.1H20c-.1 0 0 0 0-.1l-.4-1-.3 1v.1c0 .1 0 0-.1 0h-.1V74l-.2-1.5h-.1v-.2h.7v.2H19.2c-.1 0 0 1.2 0 1.2zm2-.7c0-.1 0-.1.1-.2s.1-.1.2-.1.2 0 .2.1c.1.1.1.1.1.2s0 .1-.1.2-.1.1-.2.1-.2 0-.2-.1-.1-.1-.1-.2zm0 .8c0-.1 0-.1.1-.2s.1-.1.2-.1.2 0 .2.1c.1.1.1.1.1.2s0 .1-.1.2-.1.1-.2.1-.2 0-.2-.1c0 0-.1-.1-.1-.2zm17-.8c0-.1 0-.1.1-.2s.1-.1.2-.1.2 0 .2.1.1.1.1.2 0 .1-.1.2-.1.1-.2.1-.2 0-.2-.1-.1-.1-.1-.2zm0 .8c0-.1 0-.1.1-.2s.1-.1.2-.1.2 0 .2.1.1.1.1.2 0 .1-.1.2-.1.1-.2.1-.2 0-.2-.1-.1-.1-.1-.2zm1.9-.7c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.1-.3.3-.4.5-.5zm-.1.5c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1-.1.1-.1.2-.1.3zm0-1c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1s-.2 0-.3.1c-.1.2-.1.2-.1.3zm2 .5c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.1-.3.3-.4.5-.5zm-.1.5c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1-.1.1-.1.2-.1.3zm0-1c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1-.1.2-.1.2-.1.3zm1.9.5c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.2-.3.3-.4.5-.5zm0 .5c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1s.1-.2.1-.3c0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1-.1.1-.1.2-.1.3zm0-1c0 .1 0 .2.1.2.1.2.1.2.2.2s.2 0 .3-.1c.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1s-.2 0-.3.1v.2zm1.9.5c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.2-.3.3-.4.5-.5zm0 .5c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1-.1.1-.2.2-.1.3zm0-1c0 .1 0 .2.1.2 0 .2.1.2.2.2s.2 0 .3-.1c.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1v.2zm1.9.4c0-.1 0-.1.1-.2s.1-.1.2-.1.2 0 .2.1c.1.1.1.1.1.2s0 .1-.1.2-.1.1-.2.1-.2 0-.2-.1c-.1-.1-.1-.1-.1-.2zm0 .8c0-.1 0-.1.1-.2s.1-.1.2-.1.2 0 .2.1c.1.1.1.1.1.2s0 .1-.1.2-.1.1-.2.1-.2 0-.2-.1c-.1 0-.1-.1-.1-.2zm13.2-.7c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.2-.3.3-.4.5-.5zm-.1.5c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1s.1-.2.1-.3c0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1 0 .1-.1.2-.1.3zm0-1c0 .1 0 .2.1.2.1.2.2.2.3.2.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1s-.2 0-.3.1c0 .1-.1.1-.1.2zm2 .5c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.2-.3.3-.4.5-.5zm-.1.5c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1-.1.1-.1.2-.1.3zm0-1c0 .1 0 .2.1.2.1.2.2.2.3.2.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1-.2 0-.4.1-.4.3zm2 .5c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.1-.3.3-.4.5-.5zm-.1.5c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1-.1.1-.1.2-.1.3zm0-1c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1-.1.2-.1.2-.1.3zm1.9.5c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.2-.3.4-.4.5-.5zm0 .5c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1-.1.1-.1.2-.1.3zm0-1c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1s-.2 0-.3.1c-.1.2-.1.2-.1.3zm1.9.5c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.2-.3.3-.4.5-.5zm0 .5c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1-.1.1-.1.2-.1.3zm0-1c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1s-.2 0-.3.1c-.1.2-.1.2-.1.3zm1.8.3.2-.2c.1 0 .2-.1.3-.1.4 0 .7.3.7.7 0 .2-.1.4-.2.5-.1.1-.3.2-.5.2-.1 0-.2 0-.3-.1-.1 0-.1-.1-.2-.2v.4c0 .1 0 0 0 0h-.4v-.2h.1v-1.6h-.1v-.2h.4v.8zm.5 1c.1 0 .3 0 .4-.1s.1-.2.1-.4c0-.3-.2-.5-.5-.5-.1 0-.3 0-.3.1-.1.1-.1.2-.1.4s0 .3.1.4c0 0 .2.1.3.1zm1.5-.1c0-.1 0-.1.1-.2s.1-.1.2-.1.2 0 .2.1c.1.1.1.1.1.2s0 .1-.1.2-.1.1-.2.1-.2 0-.2-.1c-.1 0-.1-.1-.1-.2zm7.5-.7c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.2-.3.3-.4.5-.5zm0 .5c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1-.1.1-.1.2-.1.3zm0-1c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1s-.2 0-.3.1c-.1.2-.1.2-.1.3zm1.9.5c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.2-.3.3-.4.5-.5zm0 .5c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1-.1.1-.1.2-.1.3zm0-1c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1-.3.1-.4.2-.4.4zm1.9.5c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.2-.3.3-.4.5-.5zm-.1.5c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1 0 .1-.1.2-.1.3zm.1-1c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1s-.2 0-.3.1c-.1.2-.1.2-.1.3zm1.9.5c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2v-.4c0-.3.1-.4.3-.5zm-.1.5c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1 0 .1-.1.2-.1.3zm0-1c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1s-.2 0-.3.1c0 .2-.1.2-.1.3zm1.9.3.2-.2c.1 0 .2-.1.3-.1.4 0 .7.3.7.7 0 .2-.1.4-.2.5-.1.1-.3.2-.5.2-.1 0-.2 0-.3-.1-.1 0-.1-.1-.2-.2v.4c0 .1 0 0 0 0h-.4v-.2h.1v-1.6h-.1v-.2h.4v.8zm.5 1c.1 0 .3 0 .4-.1.1-.1.1-.2.1-.4 0-.3-.2-.5-.5-.5-.1 0-.3 0-.3.1-.1.1-.1.2-.1.4s0 .3.1.4c0 0 .1.1.3.1zm1.5-.1c0-.1 0-.1.1-.2s.1-.1.2-.1.2 0 .2.1.1.1.1.2 0 .1-.1.2-.1.1-.2.1-.2 0-.2-.1-.1-.1-.1-.2zm5.6-.7c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.2-.3.3-.4.5-.5zm-.1.5c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1 0 .1-.1.2-.1.3zm.1-1c0 .1 0 .2.1.2 0 .2.1.2.2.2s.2 0 .3-.1c.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1s-.2 0-.3.1v.2zm1.9.5c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.2-.3.3-.4.5-.5zm-.1.5c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1s.1-.2.1-.3c0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1 0 .1-.1.2-.1.3zm0-1c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1s-.2 0-.3.1c0 .2-.1.2-.1.3zm2 .5c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.2-.3.3-.4.5-.5zm-.1.5c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1 0 .1-.1.2-.1.3zm0-1c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1 0 .2-.1.2-.1.3zm2 .5c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.1-.3.3-.4.5-.5zm-.1.5c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1-.1.1-.1.2-.1.3zm0-1c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1s-.2 0-.3.1c-.1.2-.1.2-.1.3zm2 .5c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.1-.3.3-.4.5-.5zm-.1.5c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1s.1-.2.1-.3c0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1-.1.1-.1.2-.1.3zm0-1c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1s-.2 0-.3.1c-.1.2-.1.2-.1.3zm1.8.3.2-.2c.1 0 .2-.1.3-.1.4 0 .7.3.7.7 0 .2-.1.4-.2.5-.1.1-.3.2-.5.2-.1 0-.2 0-.3-.1-.1 0-.1-.1-.2-.2v.4c0 .1 0 0 0 0h-.4v-.2h.1v-1.6h-.1v-.2h.4v.8zm.5 1c.1 0 .3 0 .4-.1.1-.1.1-.2.1-.4 0-.3-.2-.5-.5-.5-.1 0-.3 0-.3.1-.1.1-.1.2-.1.4s0 .3.1.4c.1 0 .2.1.3.1zm1.5-.1c0-.1 0-.1.1-.2s.1-.1.2-.1.2 0 .2.1.1.1.1.2 0 .1-.1.2-.1.1-.2.1-.2 0-.2-.1-.1-.1-.1-.2zm7.6 0c0-.1 0-.1.1-.2s.1-.1.2-.1.2 0 .2.1c.1.1.1.1.1.2s0 .1-.1.2-.1.1-.2.1-.2 0-.2-.1c-.1 0-.1-.1-.1-.2zm2.6.3s0-.1 0 0l-.1-.2-.2.2c-.1.1-.2.1-.3.1-.2 0-.4-.1-.5-.2-.1-.1-.2-.3-.2-.5 0-.4.3-.7.7-.7.1 0 .2 0 .3.1s.2.1.2.2v-.6H115.8v-.2h.5V74h.2v.2h-.1l-.2-.1zm-1-.7c0 .1 0 .3.1.3s.2.1.3.1c.1 0 .3 0 .3-.1.1-.1.1-.2.1-.4 0-.3-.2-.5-.5-.5-.1 0-.3 0-.3.1v.5zm2.1-.3c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.2-.3.3-.4.5-.5zm0 .5c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1-.1.1-.1.2-.1.3zm0-1c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1s-.2 0-.3.1c-.1.2-.1.2-.1.3zm1.9.5c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.2-.3.3-.4.5-.5zm-.1.5c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1s.1-.2.1-.3c0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1 0 .1-.1.2-.1.3zm.1-1c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1s-.2 0-.3.1c-.1.2-.1.2-.1.3zm1.8.3.2-.2c.1 0 .2-.1.3-.1.4 0 .7.3.7.7 0 .2-.1.4-.2.5-.1.1-.3.2-.5.2-.1 0-.2 0-.3-.1-.1 0-.1-.1-.2-.2v.4c0 .1 0 0 0 0h-.4v-.2h.1v-1.6h-.1v-.2h.4v.8zm.5 1c.1 0 .3 0 .4-.1.1-.1.1-.2.1-.4 0-.3-.2-.5-.5-.5-.1 0-.3 0-.3.1s-.1.2-.1.4 0 .3.1.4.1.1.3.1zm1.5-.1c0-.1 0-.1.1-.2s.1-.1.2-.1.2 0 .2.1.1.1.1.2 0 .1-.1.2-.1.1-.2.1-.2 0-.2-.1-.1-.1-.1-.2zm-109.4 4h-.4v-.2h.1v-1.4h-.1V76H14c.3 0 .5.1.6.2.2.2.2.4.2.7 0 .1 0 .3-.1.4 0 .1-.1.2-.2.3-.1.1-.2.1-.3.2s-.3.1-.4.1h-.1c0-.1 0-.1-.1-.1zm0-.2h.2c.3 0 .4-.1.6-.2s.2-.3.2-.5-.1-.4-.2-.5c-.1-.1-.3-.2-.6-.2h-.2v1.4zm2.8 0v-.4h-.5V77H16.8c.1 0 0 0 0 .1v.1h-.1v.6c-.1 0-.2.1-.3.1h-.3c-.3 0-.5-.1-.7-.3s-.4-.4-.4-.7.1-.5.2-.7c.2-.2.4-.3.6-.3h.2c.1 0 .1 0 .2.1v-.1h.2v.5h-.1c-.1 0-.1 0-.1-.1-.1-.2-.2-.3-.4-.3s-.3.1-.4.2c-.1.1-.2.3-.2.5s.1.4.2.6c.1.1.3.2.5.2h.2c.1.1.2.1.3.1zm1.8 0v-.4h-.5V77H18.6c.1 0 0 0 0 .1v.1h-.1v.6c-.1 0-.2.1-.3.1h-.3c-.3 0-.5-.1-.7-.3-.2-.2-.3-.4-.3-.7s.1-.5.2-.7c.2-.2.4-.3.6-.3h.2c.1 0 .1 0 .2.1v-.1h.2v.5h-.1c-.1 0-.1 0-.1-.1-.1-.2-.2-.3-.4-.3s-.3.1-.4.2c-.1.1-.2.3-.2.5s.1.4.2.6c.1.1.3.2.5.2h.2c.1.1.2.1.2.1zm1.1.2h-.4v-.2h.1v-1.4h-.1V76h.7c.3 0 .5.1.6.2.2.2.2.4.2.7 0 .1 0 .3-.1.4 0 .1-.1.2-.2.3-.1.1-.2.1-.3.2s-.3.1-.4.1h-.1c.1-.1 0-.1 0-.1zm0-.2h.2c.3 0 .4-.1.6-.2.1-.1.2-.3.2-.5s-.1-.4-.2-.5c-.1-.1-.3-.2-.6-.2h-.2v1.4zm1.9-.9c0-.1 0-.1.1-.2s.1-.1.2-.1.2 0 .2.1c.1.1.1.1.1.2s0 .1-.1.2c0 .1-.1.1-.2.1s-.2 0-.2-.1c0 0-.1-.1-.1-.2zm0 .9c0-.1 0-.1.1-.2s.1-.1.2-.1.2 0 .2.1c.1.1.1.1.1.2s0 .1-.1.2-.1.1-.2.1-.2 0-.2-.1-.1-.1-.1-.2zm17-.9c0-.1 0-.1.1-.2s.1-.1.2-.1.2 0 .2.1.1.1.1.2 0 .1-.1.2-.1.1-.2.1-.2 0-.2-.1-.1-.1-.1-.2zm0 .9c0-.1 0-.1.1-.2s.1-.1.2-.1.2 0 .2.1.1.1.1.2 0 .1-.1.2-.1.1-.2.1-.2 0-.2-.1c-.1-.1-.1-.1-.1-.2zm1.9-.7c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.1-.3.3-.5.5-.5zm-.1.4c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3s-.2 0-.3 0c-.1 0-.2 0-.3.1s-.1.1-.1.2zm0-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1s-.2 0-.3.1c-.1.1-.1.2-.1.3zm2 .5c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.1-.3.3-.5.5-.5zm-.1.4c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3h-.3c-.1 0-.2 0-.3.1-.1 0-.1.1-.1.2zm0-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1-.1.1-.1.2-.1.3zm1.9.5c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.2-.3.3-.5.5-.5zm0 .4c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3s-.2 0-.4 0c-.1 0-.2 0-.3.1s0 .1 0 .2zm0-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1s-.2 0-.3.1c-.1.1-.1.2-.1.3zm1.9.5c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.2-.3.3-.5.5-.5zm0 .4c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3s-.3 0-.4 0c-.1 0-.2 0-.3.1s0 .1 0 .2zm0-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1-.1.1-.1.2-.1.3zm1.9.3c0-.1 0-.1.1-.2s.1-.1.2-.1.2 0 .2.1c.1.1.1.1.1.2s0 .1-.1.2c0 .1-.1.1-.2.1s-.2 0-.2-.1c-.1 0-.1-.1-.1-.2zm0 .9c0-.1 0-.1.1-.2s.1-.1.2-.1.2 0 .2.1c.1.1.1.1.1.2s0 .1-.1.2-.1.1-.2.1-.2 0-.2-.1c-.1-.1-.1-.1-.1-.2zm13.2-.7c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.2-.3.3-.5.5-.5zm-.1.4c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3s-.2 0-.3 0c-.1 0-.2 0-.3.1s-.1.1-.1.2zm0-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1s-.2 0-.3.1c0 .1-.1.2-.1.3zm2 .5c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.2-.3.3-.5.5-.5zm-.1.4c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3H63c-.1 0-.2 0-.3.1s-.1.1-.1.2zm0-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1 0 .1-.1.2-.1.3zm2 .5c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.1-.3.3-.5.5-.5zm-.1.4c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3h-.3c-.1 0-.2 0-.3.1-.1 0-.1.1-.1.2zm0-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1-.1.1-.1.2-.1.3zm3.9.1c-.1 0-.1 0 0 0-.1 0-.1-.1 0 0l-.1-.7v-.1H68.6c.1 0 0 0 0 .1v.6c-.1 0-.1.1-.2.1zm.5 0s-.1 0 0 0c-.1 0-.1-.1 0 0l-.1-.7v-.1c0-.1 0 0 .1 0h.2v.7c-.1 0-.1.1-.2.1.1 0 .1 0 0 0zm1.3.4c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.2-.3.3-.5.5-.5zm0 .4c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3h-.4c-.1 0-.2 0-.3.1 0 0-.1.1 0 .2zm0-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1-.1.1-.1.2-.1.3zm1.9.5c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.2-.3.3-.5.5-.5zm-.1.4c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3h-.3c-.1 0-.2 0-.3.1 0 0-.1.1-.1.2zm0-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1s-.2 0-.3.1c0 .1-.1.2-.1.3zm1.9.3.2-.2c.1 0 .2-.1.3-.1.4 0 .7.3.7.7 0 .2-.1.4-.2.5s-.3.2-.5.2c-.1 0-.2 0-.3-.1l-.2-.2v.2c0 .1 0 0 0 0h-.4v-.2h.1v-1.6h-.1v-.2h.4v1zm.5.9c.1 0 .3 0 .4-.1.1-.1.1-.2.1-.4 0-.3-.2-.5-.5-.5-.1 0-.3 0-.3.1-.1.1-.1.2-.1.4s0 .3.1.4c0 .1.1.1.3.1zm10.9-1.1c-.1 0-.1-.1 0 0l-.1-.7v-.1H85.5c.1 0 0 0 0 .1v.6c-.1 0-.1.1-.2.1.1 0 .1 0 0 0zm.6 0c-.1 0-.1 0 0 0-.1 0-.1-.1 0 0l-.1-.7v-.1H86.1v.7c-.1 0-.1.1-.2.1zm1.3.4c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.2-.3.3-.5.5-.5zm-.1.4c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3s-.2 0-.3 0c-.1 0-.2 0-.3.1s-.1.1-.1.2zm0-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1 0 .1-.1.2-.1.3zm2 .5c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.1-.3.3-.5.5-.5zm-.1.4c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3h-.3c-.1 0-.2 0-.3.1-.1 0-.1.1-.1.2zm0-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1s-.2 0-.3.1c-.1.1-.1.2-.1.3zm1.9.3.2-.2c.1 0 .2-.1.3-.1.4 0 .7.3.7.7 0 .2-.1.4-.2.5s-.3.2-.5.2c-.1 0-.2 0-.3-.1-.1 0-.1-.1-.2-.2v.2c0 .1 0 0 0 0h-.4v-.2h.1v-1.6h-.1v-.2h.4v1zm.4.9c.1 0 .3 0 .4-.1.1-.1.1-.2.1-.4 0-.3-.2-.5-.5-.5-.1 0-.3 0-.3.1-.1.1-.1.2-.1.4s0 .3.1.4c.1.1.2.1.3.1zm3.4-.7c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.2-.3.3-.5.5-.5zm-.1.4c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3H95c-.1 0-.2 0-.3.1 0 0-.1.1-.1.2zm.1-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1s-.2 0-.3.1c-.1.1-.1.2-.1.3zm1.9.5c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4 0-.1 0-.2.1-.3.2-.1.3-.2.4-.2zm-.1.4c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3h-.3c-.1 0-.2 0-.3.1 0 0-.1.1-.1.2zm0-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1s-.2 0-.3.1c0 .1-.1.2-.1.3zm2 .5c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.2-.3.3-.5.5-.5zm-.1.4c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3s-.2 0-.3 0c-.1 0-.2 0-.3.1s-.1.1-.1.2zm0-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1 0 .1-.1.2-.1.3zm3.9.1c-.1 0-.1 0 0 0-.1 0-.1-.1 0 0l-.1-.7v-.1H102.5c.1 0 0 0 0 .1v.6c-.1 0-.1.1-.2.1zm.5 0c-.1 0-.1-.1 0 0l-.1-.7v-.1H103v.7c-.1 0-.1.1-.2.1.1 0 .1 0 0 0zm1.3.4c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.2-.3.3-.5.5-.5zm0 .4c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1s-.1.2-.1.3zm0-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1-.1.1-.1.2-.1.3zm1.9.5c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.2-.3.3-.5.5-.5zm0 .4c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1-.1.1-.1.2-.1.3zm0-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1s-.2 0-.3.1c-.1.1-.1.2-.1.3zm1.8.3.2-.2c.1 0 .2-.1.3-.1.4 0 .7.3.7.7 0 .2-.1.4-.2.5s-.3.2-.5.2c-.1 0-.2 0-.3-.1-.1 0-.1-.1-.2-.2v.2c0 .1 0 0 0 0h-.4v-.2h.1v-1.6h-.1v-.2h.4v1zm.5.9c.1 0 .3 0 .4-.1.1-.1.1-.2.1-.4 0-.3-.2-.5-.5-.5-.1 0-.3 0-.3.1-.1.1-.1.2-.1.4s0 .3.1.4c0 .1.2.1.3.1zm4.1.2-.1-.2-.2.2c-.1 0-.2.1-.3.1-.2 0-.4-.1-.5-.2-.1-.1-.2-.3-.2-.5 0-.4.3-.7.7-.7.1 0 .2 0 .3.1.1 0 .2.1.2.2v-.6H112V76h.5v1.7h.2v.2h-.1l-.2-.1zm-1-.6c0 .1 0 .3.1.3s.2.1.3.1.3 0 .3-.1.1-.2.1-.4c0-.3-.2-.5-.5-.5-.1 0-.3 0-.3.1v.5zm2.2-.3c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.1-.3.3-.5.5-.5zm-.1.4c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1-.1.1-.1.2-.1.3zm0-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1s-.2 0-.3.1c-.1.1-.1.2-.1.3zm1.9.5c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.2-.3.3-.5.5-.5zm0 .4c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1-.1.1-.1.2-.1.3zm0-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1-.1.1-.1.2-.1.3zm1.9.1s0-.1 0 0l-.1-.7v-.1c0-.1 0 0 .1 0h.2c.1 0 0 0 0 .1v.6s-.1.1-.2.1c.1 0 .1 0 0 0zm.6 0s-.1 0 0 0c-.1 0-.1-.1 0 0l-.1-.7v-.1c0-.1 0 0 .1 0h.2c.1 0 0 0 0 .1v.6c-.1 0-.1.1-.2.1.1 0 0 0 0 0zm1.3 0s0-.1 0 0l-.1-.7v-.1c0-.1 0 0 .1 0h.2c.1 0 0 0 0 .1v.6c-.1 0-.1.1-.2.1.1 0 .1 0 0 0zm.6 0c-.1 0-.1 0 0 0-.1 0-.1-.1 0 0l-.1-.7v-.1H120v.7c-.1 0-.1.1-.2.1.1 0 0 0 0 0zm1.3.4c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.2-.3.3-.5.5-.5zm-.1.4c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1 0 .1-.1.2-.1.3zm0-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1 0 .1-.1.2-.1.3zm2 .5c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.2-.3.3-.5.5-.5zm-.1.4c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1-.1.1-.1.2-.1.3zm0-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1s-.2 0-.3.1c0 .1-.1.2-.1.3zm1.9.3.2-.2c.1 0 .2-.1.3-.1.4 0 .7.3.7.7 0 .2-.1.4-.2.5s-.3.2-.5.2c-.1 0-.2 0-.3-.1-.1 0-.1-.1-.2-.2v.2c0 .1 0 0 0 0h-.4v-.2h.1v-1.6h-.1v-.2h.4v1zm.5.9c.1 0 .3 0 .4-.1.1-.1.1-.2.1-.4 0-.3-.2-.5-.5-.5-.1 0-.3 0-.3.1-.1.1-.1.2-.1.4s0 .3.1.4c0 .1.1.1.3.1zm-87.1 2.9c0-.1 0-.1.1-.2s.1-.1.2-.1.2 0 .2.1.1.1.1.2 0 .1-.1.2-.1.1-.2.1-.2 0-.2-.1-.1-.1-.1-.2zm0 .9c0-.1 0-.1.1-.2s.1-.1.2-.1.2 0 .2.1.1.1.1.2 0 .1-.1.2-.1.1-.2.1-.2 0-.2-.1-.1-.2-.1-.2zm1.9-.8c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.1-.2.3-.4.5-.5zm-.1.5c0 .1 0 .2.1.3s.2.1.3.1c.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3s-.2-.1-.3-.1c-.1 0-.2 0-.3.1s-.1.2-.1.3zm0-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1s-.2 0-.3.1c-.1.1-.1.2-.1.3zm2 .4c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.1-.2.3-.4.5-.5zm-.1.5c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1-.1.1-.1.2-.1.3zm0-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1-.1.1-.1.2-.1.3zm1.9.4c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.2-.2.3-.4.5-.5zm0 .5c0 .1 0 .2.1.3s.2.1.3.1.2 0 .3-.1.1-.2.1-.3c0-.1 0-.2-.1-.3s-.2-.1-.3-.1c-.1 0-.2 0-.3.1s-.1.2-.1.3zm0-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1s-.2 0-.3.1c-.1.1-.1.2-.1.3zm1.9.4c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.2-.2.3-.4.5-.5zm0 .5c0 .1 0 .2.1.3s.2.1.3.1c.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3s-.2-.1-.3-.1c-.1 0-.2 0-.3.1s-.1.2-.1.3zm0-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1-.1.1-.1.2-.1.3zm1.9.3c0-.1 0-.1.1-.2s.1-.1.2-.1.2 0 .2.1c.1.1.1.1.1.2s0 .1-.1.2-.1.1-.2.1-.2 0-.2-.1c-.1 0-.1-.1-.1-.2zm0 .9c0-.1 0-.1.1-.2s.1-.1.2-.1.2 0 .2.1c.1.1.1.1.1.2s0 .1-.1.2-.1.1-.2.1-.2 0-.2-.1c-.1-.1-.1-.2-.1-.2zm13.2-.8c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.2-.2.3-.4.5-.5zm-.1.5c0 .1 0 .2.1.3s.2.1.3.1.2 0 .3-.1.1-.2.1-.3c0-.1 0-.2-.1-.3s-.2-.1-.3-.1c-.1 0-.2 0-.3.1s-.1.2-.1.3zm0-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1s-.2 0-.3.1c0 .1-.1.2-.1.3zm2 .4c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.2-.2.3-.4.5-.5zm-.1.5c0 .1 0 .2.1.3s.2.1.3.1c.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1s-.1.2-.1.3zm0-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1 0 .1-.1.2-.1.3zm2 .4c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.1-.2.3-.4.5-.5zm-.1.5c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1-.1.1-.1.2-.1.3zm0-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1-.1.1-.1.2-.1.3zm5.7.4c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.2-.2.3-.4.5-.5zm0 .5c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1-.1.1-.2.2-.1.3zm0-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1-.1.1-.1.2-.1.3zm1.9.4c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.2.2.2.3.2.5 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1 0 .1.1.1.2 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.2-.2.3-.4.5-.5zm-.1.5c0 .1 0 .2.1.3s.2.1.3.1c.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1 0 .1-.1.2-.1.3zm0-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1s-.2 0-.3.1c0 .1-.1.2-.1.3zm2 .4c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.2-.2.3-.4.5-.5zm-.1.5c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1 0 .1-.1.2-.1.3zm0-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1s-.2 0-.3.1c0 .1-.1.2-.1.3zm3.9 1.2c0-.1 0-.1.1-.2s.1-.1.2-.1.2 0 .2.1c.1.1.1.1.1.2s0 .1-.1.2-.1.1-.2.1-.2 0-.2-.1c-.1-.1-.1-.2-.1-.2zm2.6.2s-.1 0 0 0l-.1-.2-.2.2c-.1 0-.2.1-.3.1-.2 0-.4-.1-.5-.2-.1-.1-.2-.3-.2-.5 0-.4.3-.7.7-.7.1 0 .2 0 .3.1s.2.1.2.2V80H80v-.2h.5v1.7h.2v.2h-.1l-.2-.1zm-1-.7c0 .1 0 .3.1.3s.2.1.3.1.3 0 .3-.1.1-.2.1-.4c0-.3-.2-.5-.5-.5-.1 0-.3 0-.3.1v.5zm2.1-.3c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.2-.2.3-.4.5-.5zm0 .5c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3s-.2-.1-.3-.1c-.1 0-.2 0-.3.1s-.1.2-.1.3zm0-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1-.1.1-.1.2-.1.3zm1.9.4c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.2-.2.3-.4.5-.5zm-.1.5c0 .1 0 .2.1.3s.2.1.3.1c.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1 0 .1-.1.2-.1.3zm.1-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1s-.2 0-.3.1c-.1.1-.1.2-.1.3zm1.9.4c-.1 0-.2-.1-.2-.2-.1 0-.1-.1-.1-.2 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2v-.5c0-.2.1-.4.3-.5zm-.1.5c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1 0 .1-.1.2-.1.3zm0-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1s-.2 0-.3.1c0 .1-.1.2-.1.3zm2 .4c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.2-.2.3-.4.5-.5zm-.1.5c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3s-.2-.1-.3-.1c-.1 0-.2 0-.3.1s-.1.2-.1.3zm0-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1 0 .1-.1.2-.1.3zm2 .4c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.1-.2.3-.4.5-.5zm-.1.5c0 .1 0 .2.1.3s.2.1.3.1c.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1-.1.1-.1.2-.1.3zm0-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1s-.2 0-.3.1c-.1.1-.1.2-.1.3zm1.9.4c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4l.5-.5zm0 .5c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1s.1-.2.1-.3c0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1-.1.1-.1.2-.1.3zm0-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1s-.2 0-.3.1c-.1.1-.1.2-.1.3zm3.8.4c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.2-.2.3-.4.5-.5zm-.1.5c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1 0 .1-.1.2-.1.3zm.1-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1s-.2 0-.3.1c-.1.1-.1.2-.1.3zm1.9.4c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.2-.2.3-.4.5-.5zm-.1.5c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1s.1-.2.1-.3c0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1 0 .1-.1.2-.1.3zm0-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1s-.2 0-.3.1c0 .1-.1.2-.1.3zm2 .4c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.2-.2.3-.4.5-.5zm-.1.5c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3s-.2-.1-.3-.1c-.1 0-.2 0-.3.1s-.1.2-.1.3zm0-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1 0 .1-.1.2-.1.3zm5.7.4c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.2-.2.3-.4.5-.5zm0 .5c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1s-.1.2-.1.3zm0-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1-.1.1-.1.2-.1.3zm1.9.4c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.2-.2.3-.4.5-.5zm0 .5c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1-.1.1-.1.2-.1.3zm0-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1s-.2 0-.3.1c-.1.1-.1.2-.1.3zm1.9.4c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.2-.2.3-.4.5-.5zm-.1.5c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1s.1-.2.1-.3c0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1 0 .1-.1.2-.1.3zm.1-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1s-.2 0-.3.1c-.1.1-.1.2-.1.3zm3.8.4c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.2-.2.3-.4.5-.5zm-.1.5c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1-.1.1-.1.2-.1.3zm0-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1s-.2 0-.3.1c0 .1-.1.2-.1.3zm2 .4c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.1-.2.3-.4.5-.5zm-.1.5c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1s.1-.2.1-.3c0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1-.1.1-.1.2-.1.3zm0-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1s-.2 0-.3.1c-.1.1-.1.2-.1.3zm1.9.4c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.2-.2.3-.4.5-.5zm0 .5c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1-.1.1-.1.2-.1.3zm0-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1-.1.1-.1.2-.1.3zm5.7.4c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.2-.2.3-.4.5-.5zm-.1.5c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1 0 .1-.1.2-.1.3zm0-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1 0 .1-.1.2-.1.3zm2 .4c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.2-.2.3-.4.5-.5zm-.1.5c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1-.1.1-.1.2-.1.3zm0-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1s-.2 0-.3.1c0 .1-.1.2-.1.3zm2 .4c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.1-.2.3-.4.5-.5zm-.1.5c0 .1 0 .2.1.3s.2.1.3.1c.1 0 .2 0 .3-.1s.1-.2.1-.3c0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1-.1.1-.1.2-.1.3zm0-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1s-.2 0-.3.1c-.1.1-.1.2-.1.3zm-86.6 4.1c0-.1 0-.1.1-.2s.1-.1.2-.1.2 0 .2.1.1.1.1.2 0 .1-.1.2-.1.1-.2.1-.2 0-.2-.1-.1-.1-.1-.2zm0 .8c0-.1 0-.1.1-.2s.1-.1.2-.1.2 0 .2.1.1.1.1.2 0 .1-.1.2-.1.1-.2.1-.2 0-.2-.1-.1-.1-.1-.2zM40 85l.3-.9V84H40.6v.1l.3.9.2-1.2H40.8v-.2h.7v.2h-.1l-.4 1.5v.1h-.2v-.1l-.3-1.1-.3 1.1v.1H40v-.1l-.2-1.5h-.1v-.2h.4v.2H39.8L40 85zm2-.6c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.1-.2.3-.4.5-.5zm-.1.5c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1-.1.1-.1.2-.1.3zm0-1c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1-.1.2-.1.3-.1.3zm1.9.5c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.2-.2.3-.4.5-.5zm0 .5c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1-.1.1-.1.2-.1.3zm0-1c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1s-.2 0-.3.1c-.1.2-.1.3-.1.3zm1.9.5c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.2-.2.3-.4.5-.5zm0 .5c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1-.1.1-.2.2-.1.3zm0-1c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1-.1.2-.1.3-.1.3zm1.9.4c0-.1 0-.1.1-.2s.1-.1.2-.1.2 0 .2.1c.1.1.1.1.1.2s0 .1-.1.2-.1.1-.2.1-.2 0-.2-.1c-.1-.1-.1-.1-.1-.2zm0 .8c0-.1 0-.1.1-.2s.1-.1.2-.1.2 0 .2.1c.1.1.1.1.1.2s0 .1-.1.2-.1.1-.2.1-.2 0-.2-.1c-.1 0-.1-.1-.1-.2zm13.2-.7c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.2-.2.3-.4.5-.5zm-.1.5c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1 0 .1-.1.2-.1.3zm0-1c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1s-.2 0-.3.1c0 .2-.1.3-.1.3zm2 .5c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.2-.2.3-.4.5-.5zm-.1.5c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1-.1.1-.1.2-.1.3zm0-1c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1 0 .2-.1.3-.1.3zm2 .5c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.1-.2.3-.4.5-.5zm-.1.5c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1-.1.1-.1.2-.1.3zm0-1c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1-.1.2-.1.3-.1.3zm5.7.5c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.2-.2.3-.4.5-.5zm0 .5c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1-.1.1-.2.2-.1.3zm0-1c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1-.1.2-.1.3-.1.3zm1.9.5c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.2-.2.3-.4.5-.5zm-.1.5c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1 0 .1-.1.2-.1.3zm0-1c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1s-.2 0-.3.1c0 .2-.1.3-.1.3zm2 .5c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.2-.2.3-.4.5-.5zm-.1.5c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1 0 .1-.1.2-.1.3zm0-1c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1s-.2 0-.3.1c0 .2-.1.3-.1.3zm3.9.5c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.1-.2.3-.4.5-.5zm-.1.5c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1-.1.1-.1.2-.1.3zm0-1c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1s-.2 0-.3.1c-.1.2-.1.3-.1.3zm1.9.5c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.2-.2.3-.4.5-.5zm0 .5c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1-.1.1-.1.2-.1.3zm0-1c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1s-.2 0-.3.1c-.1.2-.1.3-.1.3zm1.9.5c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.2-.2.3-.4.5-.5zm0 .5c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1-.1.1-.1.2-.1.3zm0-1c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1-.1.2-.1.3-.1.3zm5.7.5c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.2-.2.3-.4.5-.5zm-.1.5c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1-.1.1-.1.2-.1.3zm0-1c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1 0 .2-.1.3-.1.3zm2 .5c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.1-.2.3-.4.5-.5zm-.1.5c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1-.1.1-.1.2-.1.3zm0-1c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1s-.2 0-.3.1c-.1.2-.1.3-.1.3zm1.9.5c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4l.5-.5zm0 .5c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1-.1.1-.1.2-.1.3zm0-1c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1s-.2 0-.3.1c-.1.2-.1.3-.1.3zm3.8.5c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.2-.2.3-.4.5-.5zm-.1.5c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1 0 .1-.1.2-.1.3zm.1-1c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1s-.2 0-.3.1c-.1.2-.1.3-.1.3zm1.9.5c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.2-.2.3-.4.5-.5zm-.1.5c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1 0 .1-.1.2-.1.3zm0-1c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1s-.2 0-.3.1c0 .2-.1.3-.1.3zm2 .5c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.2-.2.3-.4.5-.5zm-.1.5c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1 0 .1-.1.2-.1.3zm0-1c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1 0 .2-.1.3-.1.3zm5.7.5c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.2-.2.3-.4.5-.5zm0 .5c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1-.1.1-.1.2-.1.3zm0-1c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1-.1.2-.1.3-.1.3zm1.9.5c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.2-.2.3-.4.5-.5zm0 .5c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1-.1.1-.1.2-.1.3zm0-1c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1s-.2 0-.3.1c-.1.2-.1.3-.1.3zm1.9.5c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.2-.2.3-.4.5-.5zm-.1.5c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1 0 .1-.1.2-.1.3zm.1-1c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1s-.2 0-.3.1c-.1.2-.1.3-.1.3zm4.4 1.3h.2c.1 0 0 0 0 .1v.1H111.6v-.2H111.9v-.5l-.5-.9h-.1v-.2h.6v.2h-.2l.4.7.4-.7h-.2v-.2h.5v.2h-.1l-.5.9v.5h.1zm1.3-.8c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.1-.2.3-.4.5-.5zm-.1.5c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1-.1.1-.1.2-.1.3zm0-1c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1s-.2 0-.3.1c-.1.2-.1.3-.1.3zm1.9.5c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.2-.2.3-.4.5-.5zm0 .5c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1-.1.1-.1.2-.1.3zm0-1c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1-.1.2-.1.3-.1.3zm1.9 1.2c0-.1 0-.1.1-.2s.1-.1.2-.1.2 0 .2.1.1.1.1.2 0 .1-.1.2-.1.1-.2.1-.2 0-.2-.1-.1-.1-.1-.2zm1.9 0c0-.1 0-.1.1-.2s.1-.1.2-.1.2 0 .2.1c.1.1.1.1.1.2s0 .1-.1.2-.1.1-.2.1-.2 0-.2-.1c-.1 0-.1-.1-.1-.2zm1.9-.7c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.2-.2.3-.4.5-.5zm-.1.5c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1 0 .1-.1.2-.1.3zm0-1c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1 0 .2-.1.3-.1.3zm2 .5c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.2-.2.3-.4.5-.5zm-.1.5c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1-.1.1-.1.2-.1.3zm0-1c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1s-.2 0-.3.1c0 .2-.1.3-.1.3zm2.3 1.3h.1v.2h-1v-.2H124.6v-1.4H124.3v-.2H125.1c.2 0 .4 0 .6.1.1.1.2.2.2.4s-.1.3-.2.4c-.1.1-.3.1-.6.1h-.3v.5l.4.1zm-.3-1.4v.7h.3c.2 0 .3 0 .4-.1s.1-.1.1-.3c0-.1 0-.2-.1-.3s-.2-.1-.4-.1c0 .1-.3.1-.3.1zm-86.7 4.3c0-.1 0-.1.1-.2s.1-.1.2-.1.2 0 .2.1.1.1.1.2 0 .1-.1.2-.1.1-.2.1-.2 0-.2-.1-.1-.2-.1-.2zm0 .8c0-.1 0-.1.1-.2s.1-.1.2-.1.2 0 .2.1.1.1.1.2 0 .1-.1.2-.1.1-.2.1-.2 0-.2-.1-.1-.1-.1-.2zm1.9-.7c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.1-.3.3-.5.5-.5zm-.1.4c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3s-.2-.1-.3-.1c-.1 0-.2 0-.3.1s-.1.2-.1.3zm0-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1s-.2 0-.3.1c-.1.1-.1.2-.1.3zm2 .5c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.1-.3.3-.5.5-.5zm-.1.4c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1-.1.1-.1.2-.1.3zm0-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1-.1.1-.1.2-.1.3zm1.9.5c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.2-.3.3-.5.5-.5zm0 .4c0 .1 0 .2.1.3 0 .1.1.1.2.1s.2 0 .3-.1c.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3s-.2-.1-.3-.1c-.1 0-.2 0-.3.1s0 .2 0 .3zm0-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1s-.2 0-.3.1c-.1.1-.1.2-.1.3zm1.9.5c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.2-.3.3-.5.5-.5zm0 .4c0 .1 0 .2.1.3 0 .1.1.1.2.1s.2 0 .3-.1c.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3s-.2-.1-.3-.1c-.1 0-.2 0-.3.1s0 .2 0 .3zm0-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1-.1.1-.1.2-.1.3zm1.9.4c0-.1 0-.1.1-.2s.1-.1.2-.1.2 0 .2.1c.1.1.1.1.1.2s0 .1-.1.2-.1.1-.2.1-.2 0-.2-.1c-.1-.1-.1-.2-.1-.2zm0 .8c0-.1 0-.1.1-.2s.1-.1.2-.1.2 0 .2.1c.1.1.1.1.1.2s0 .1-.1.2-.1.1-.2.1-.2 0-.2-.1c-.1-.1-.1-.1-.1-.2zm13.2-.7c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.2-.3.3-.5.5-.5zm-.1.4c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3s-.2-.1-.3-.1c-.1 0-.2 0-.3.1s-.1.2-.1.3zm0-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1s-.2 0-.3.1c0 .1-.1.2-.1.3zm2 .5c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.2-.3.3-.5.5-.5zm-.1.4c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1s-.1.2-.1.3zm0-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1 0 .1-.1.2-.1.3zm2 .5c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.1-.3.3-.5.5-.5zm-.1.4c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1-.1.1-.1.2-.1.3zm0-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1-.1.1-.1.2-.1.3zm5.7.5c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.2-.3.3-.5.5-.5zm0 .4c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1-.1.1-.2.2-.1.3zm0-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1-.1.1-.1.2-.1.3zm1.9.5c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.2-.2-.3-.3c-.1-.1-.2-.3-.2-.4 0-.2.1-.4.3-.4zm-.1.4c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1 0 .1-.1.2-.1.3zm0-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1s-.2 0-.3.1c0 .1-.1.2-.1.3zm2 .5c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.2-.2-.4-.3c-.1-.1-.2-.3-.2-.4.1-.2.2-.4.4-.4zm-.1.4c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1 0 .1-.1.2-.1.3zm0-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1s-.2 0-.3.1c0 .1-.1.2-.1.3zm3.9.1c-.1 0-.1 0 0 0-.1 0-.1 0 0 0l-.1-.7V87c0-.1 0 0 .1 0h.2c.1 0 0 0 0 .1v.6c-.1.1-.1.1-.2.1.1 0 0 0 0 0zm.5 0c-.1 0-.1 0 0 0l-.1-.7V87H78.5c.1 0 0 0 0 .1v.6c-.1.1-.1.1-.2.1.1 0 .1 0 0 0zm2 1.1h.2c.1 0 0 0 0 .1v.1H79.6v-.2H79.9v-.5l-.5-.9h-.1v-.2h.6v.2h-.2l.4.7.4-.7h-.2v-.2h.5v.2h-.1l-.5.9v.5h.1zm1.2-.7c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.2-.3.3-.5.5-.5zm0 .4c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3s-.2-.1-.3-.1c-.1 0-.2 0-.3.1s-.1.2-.1.3zm0-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1-.1.1-.1.2-.1.3zm1.9.5c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.2-.3.3-.5.5-.5zm-.1.4c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1 0 .1-.1.2-.1.3zm.1-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1s-.2 0-.3.1c-.1.1-.1.2-.1.3zm1.9.5c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2v-.5c0-.2.1-.4.3-.4zm-.1.4c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1 0 .1-.1.2-.1.3zm0-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1s-.2 0-.3.1c0 .1-.1.2-.1.3zm2 .5c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.2-.3.3-.5.5-.5zm-.1.4c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3s-.2-.1-.3-.1c-.1 0-.2 0-.3.1s-.1.2-.1.3zm0-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1 0 .1-.1.2-.1.3zm2 .5c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.2-.4-.3c-.1-.1-.2-.3-.2-.4 0-.2.2-.4.4-.4zm-.1.4c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1-.1.1-.1.2-.1.3zm0-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1s-.2 0-.3.1c-.1.1-.1.2-.1.3zm1.9.5c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.2-.3.4-.5.5-.5zm0 .4c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1-.1.1-.1.2-.1.3zm0-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1s-.2 0-.3.1c-.1.1-.1.2-.1.3zm3.8.5c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.2-.3.3-.5.5-.5zm-.1.4c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1 0 .1-.1.2-.1.3zm.1-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1s-.2 0-.3.1c-.1.1-.1.2-.1.3zm1.9.5c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.2-.3.3-.5.5-.5zm-.1.4c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1 0 .1-.1.2-.1.3zm0-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1s-.2 0-.3.1c0 .1-.1.2-.1.3zm2 .5c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.2-.3.3-.5.5-.5zm-.1.4c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3s-.2-.1-.3-.1c-.1 0-.2 0-.3.1s-.1.2-.1.3zm0-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1 0 .1-.1.2-.1.3zm5.7.5c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.2-.3.3-.5.5-.5zm0 .4c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1s-.1.2-.1.3zm0-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1-.1.1-.1.2-.1.3zm1.9.5c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.2-.3.3-.5.5-.5zm0 .4c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1-.1.1-.1.2-.1.3zm0-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1s-.2 0-.3.1c-.1.1-.1.2-.1.3zm1.9.5c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.2-.3.3-.5.5-.5zm-.1.4c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1 0 .1-.1.2-.1.3zm.1-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1s-.2 0-.3.1c-.1.1-.1.2-.1.3zm5.7.1c-.1 0-.1 0 0 0-.1 0-.1 0 0 0l-.1-.7V87c0-.1 0 0 .1 0h.2c.1 0 0 0 0 .1v.6c-.1.1-.1.1-.2.1.1 0 0 0 0 0zm.5 0s-.1 0 0 0l-.1-.7V87H114.3v.7c-.1.1-.1.1-.2.1.1 0 .1 0 0 0zm2 1.1h.2c.1 0 0 0 0 .1v.1H115.4v-.2H115.7v-.5l-.5-.9h-.1v-.2h.6v.2h-.2l.4.7.4-.7h-.2v-.2h.5v.2h-.1l-.5.9v.5h.1zm1.2-.7c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.2-.3.3-.5.5-.5zm0 .4c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1-.1.1-.1.2-.1.3zm0-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1s-.2 0-.3.1c-.1.1-.1.2-.1.3zm1.9.5c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.2-.3.3-.5.5-.5zm-.1.4c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1 0 .1-.1.2-.1.3zm.1-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1s-.2 0-.3.1c-.1.1-.1.2-.1.3zm2.3 1.2h.1v.2h-1v-.2H120.9v-1.4H120.6v-.2H121.4c.2 0 .4 0 .6.1.1.1.2.2.2.4s-.1.3-.2.4c-.1.1-.3.1-.6.1h-.3v.5l.4.1zm-.4-1.3v.7h.3c.2 0 .3 0 .4-.1.1-.1.1-.1.1-.3 0-.1 0-.2-.1-.3s-.2-.1-.4-.1c0 .1-.3.1-.3.1zm1.9.2c-.1 0-.1 0 0 0l-.1-.7V87c0-.1 0 0 .1 0h.2c.1 0 0 0 0 .1v.6c-.1.1-.1.1-.2.1.1 0 .1 0 0 0zm.5 0-.1-.7V87c0-.1 0 0 .1 0h.2c.1 0 0 0 0 .1v.6c-.1.1-.1.1-.2.1.1 0 .1 0 0 0zm-83.8 5.1h-.2v-.2H39.8v-1.4h-.2v-.2H41.1v.6h-.2v-.4h-.8v.5h.4v-.2h.2v.5h-.2V92h-.4v.7h.8v-.4h.2v.6H39.7zm2.3-1c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.1-.2.3-.4.5-.5zm-.1.5c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1-.1.1-.1.2-.1.3zm0-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1-.1.1-.1.2-.1.3zm1.9.4c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.2-.2.3-.4.5-.5zm0 .5c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1-.1.1-.1.2-.1.3zm0-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1s-.2 0-.3.1c-.1.1-.1.2-.1.3zm1.9.4c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4 0 .1-.2.2-.4.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.1-.2.2-.4.4-.5zm0 .5c0 .1 0 .2.1.3.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3-.2 0-.3-.1-.4-.1-.1 0-.2 0-.3.1 0 .1-.1.2 0 .3zm0-.9c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1-.1.1-.1.2-.1.3zm2.4 1.2h.6v.2H47.3v-.2h.6v-1h-.4v-.2h.6v1.2zm-.3-1.6v-.2h.2v.4c0 .1-.1 0-.1 0h-.1v-.2zm-7 4.3h.1v.2h-.7v.5c0 .2 0 .3.1.3s.1.1.2.1.2 0 .3-.1.2-.1.2-.1h.1v.2l-.1.1c-.1 0-.2.1-.3.1h-.2c-.1 0-.2 0-.3-.1-.1 0-.1-.1-.2-.2v-.9h-.4v-.2h.4v-.4h.2v.5h.6zm1.1.9.3-.9v-.1H42.5v.1l.3.9.2-1.2H42.7v-.2h.7v.2h-.1l-.2 1.5v.1h-.2v-.1l-.3-1.1-.6 1.1v.1h-.2v-.1l-.2-1.5h-.1v-.2h.7v.2H41.9v1.2zm1.9-.6c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.2-.2.3-.4.5-.5zm0 .5c0 .1 0 .2.1.3s.2.1.3.1.2 0 .3-.1.1-.2.1-.3c0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1-.1.1-.1.2-.1.3zm0-1c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1s-.2 0-.3.1c-.1.2-.1.3-.1.3zm1.9.5c-.1 0-.2-.1-.2-.2-.1-.1-.1-.2-.1-.3 0-.2.1-.3.2-.4.1-.1.3-.2.4-.2.2 0 .3.1.4.2.1.1.2.2.2.4 0 .1 0 .2-.1.3l-.2.2c.1 0 .2.1.2.2.1.1.1.2.1.3 0 .2-.1.3-.2.4-.1.1-.3.2-.5.2s-.3-.1-.5-.2c-.1-.1-.2-.3-.2-.4.2-.2.3-.4.5-.5zm0 .5c0 .1 0 .2.1.3s.2.1.3.1c.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1-.1.1-.2.2-.1.3zm0-1c0 .1 0 .2.1.2.1.1.2.1.3.1.1 0 .2 0 .3-.1.1-.1.1-.2.1-.3 0-.1 0-.2-.1-.2-.1-.1-.2-.1-.3-.1-.1 0-.2 0-.3.1-.1.2-.1.3-.1.3zm1.8 1.5h-.4v-.2h.1v-1.4h-.1v-.2h.7c.3 0 .5.1.6.2.2.2.2.4.2.7 0 .1 0 .3-.1.4 0 .1-.1.2-.2.3-.1.1-.2.1-.3.2-.1 0-.3.1-.4.1h-.1c.1-.1.1-.1 0-.1zm0-.2h.2c.3 0 .4-.1.6-.2.1-.1.2-.3.2-.5s-.1-.4-.2-.5c-.1-.1-.3-.2-.6-.2h-.2v1.4z"/>
</svg>

```